Python 2.7.6 (default, Jun , Pyfusion 0.6.6b (d9236cf) Wed Apr 20 14:40:15 2016
 0 0.48 examples/plot_specgram.py     : ) ConfigParser.NoSectionError: No section: 'Diagnostic:' 
 1 0.478 examples/get_basic_params.py  : se ValueError('no good shots') ValueError: no good shots 
 2 1.09 examples/tables_extract.py    : OK!
 3 0.312 examples/slider_f_t.py        : ror: [Errno 2] No such file or directory: 'MP512all.npz' 
 4 1.688 examples/check_HD_histogram.py: OK!
 5 1.323 examples/prepfs_range_mp_2.py : OK!
 6 0.353 examples/merge_h1_diagnostics.py:  t_mid = dd['t_mid'] NameError: name 'dd' is not defined 
 7 0.479 examples/mode_identify_script_temp.py:  mode = mode_list[0] IndexError: list index out of range 
 8 0.612 examples/barber_pole.py       : OK!
 9 0.013 examples/example_fragments.py : overlap=220                ^ SyntaxError: invalid syntax 
10 0.479 examples/mode_identify_script.py:  mode = mode_list[0] IndexError: list index out of range 
11 0.156 examples/DA_troubles.py       : OK!
12 0.182 examples/setup_pyfusion.py    : OK!
13 0.279 examples/test_lasso_fs.py     : or key in dd.keys(): NameError: name 'dd' is not defined 
14 1.555 examples/medium_300.py        : OK!
15 0.013 examples/mode_identify_example_from_ICPP_2010.py: ll.txt' skip=4             ^ SyntaxError: invalid syntax 
16 7.458 examples/prepfs_range.py      : OK!
17 4.472 examples/gen_fs_local.py      : OK!
18 2.427 examples/plot_svd.py          : OK!
19 0.492 examples/process_swept_Langmuir_2.py:  the url is not available<type 'exceptions.LookupError'> 
20 0.928 examples/DP14_mode_plot.py    : OK!
21 0.012 examples/W7X_analysis_Fri_11Mar.py: sharey=2                   ^ SyntaxError: invalid syntax 
22 0.012 examples/compare_bands.py     : ption=()                   ^ SyntaxError: invalid syntax 
23 0.49 examples/mode_identify_script_old.py: ue') ValueError: Need to choose doN=True and/or doM=True 
24 0.505 examples/check_W7X_timebase.py:  the url is not available<type 'exceptions.LookupError'> 
25 0.487 examples/process_swept_Langmuir_1.py:  the url is not available<type 'exceptions.LookupError'> 
26 1.094 examples/plot_signals.py      : OK!
27 2.137 examples/trace_cluster.py     : kh','freq',ind=w23) NameError: name 'w23' is not defined 
28 0.345 examples/copy_DA.py           : ./datamining/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' 
29 0.017 examples/labelled_N_modes.py  : _list=None                 ^ SyntaxError: invalid syntax 
30 0.519 examples/process_swept_Langmuir.py:  the url is not available<type 'exceptions.LookupError'> 
31 7.259 examples/h1_mode_lists.py     : OK!
32 2.244 examples/cluster_info.py      : OK!
33 6.797 examples/gen_fs_bands.py      : :  ########shot 27233: 230/1428 fs finished in 5.17/6.04 
34 2.542 examples/phase_histogram.py   : OK!
35 0.477 examples/find_last_data.py    : .format(l=low))  ValueError: low limit of None not there 
36 1.168 examples/check_HMA_PMA_phase.py: OK!
37 2.462 examples/plot_svd1.py         : OK!
38 0.414 examples/find_close_shot.py   : OK!
39 0.523 examples/realtime_mode.py     : e also cm.gray_r etc NameError: name 'cm' is not defined 
40 0.284 examples/process_chirps.py    :  dfdtunit=dfdtunit)  NameError: name 'dd' is not defined 
41 1.639 examples/cross_phase.py       : OK!
42 0.292 examples/check_DA.py          : /datamining/old/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' 
43 0.059 examples/prepfs_time_mp.py    :  output_path prepfs_time_mp.py: error: too few arguments 
44 2.434 examples/plot_svd_simple.py   : OK!
45 0.284 examples/plot_many.py         : NOT AVAILABLE!!!!!') ValueError: SCAN NOT AVAILABLE!!!!! 
46 0.175 examples/query_h1ds.py        : soup import SqlSoup ImportError: No module named sqlsoup 
47 2.476 examples/plot_basic.py        : vide by zero encountered in true_divide   1/(dy1+dy2),)) 
48 0.506 examples/test_W7X_LP1.py      :  the url is not available<type 'exceptions.LookupError'> 
49 5.651 examples/cluster_phases_three_colour.py: to [10 11 12 13 14 15] - i.e. cldata has different sel,  
50 0.313 examples/new_mode_identify_script.py: ue') ValueError: Need to choose doN=True and/or doM=True 
51 3.599 examples/clean_up_pyfusion_text.py: OK!
52 0.332 examples/test_args.py         : OK!
53 0.866 examples/check_other_data.py  : OK!
54 0.012 examples/find_VSL_locking.py  : ist=mmm                    ^ SyntaxError: invalid syntax 
55 2.67 examples/cluster_DA_0.py      : OK!
56 0.289 examples/plot_text_fft.py     : r: [Errno 2] No such file or directory: 'g:fft26_50.dat' 
57 0.582 examples/merge_text_pyfusion.py: OK!
58 1.07 examples/multi_plot.py        : : igetfile None/wp/wp@105396.dat or .gz, .bz2 not found. 
59 2.178 examples/channel_spectra.py   : OK!
60 0.012 examples/shauns_clustering.py :         ^ SyntaxError: EOL while scanning string literal 
61 0.18 examples/plot_json_timeseries.py: OK!
62 0.298 examples/correct_VSL.py       : ppend_to_DA ImportError: cannot import name append_to_DA 
63 0.307 examples/fft_magnitude_extraction.py:  import HMA_funcs ImportError: No module named HMA_funcs 
64 0.014 examples/good_ones_density_cluster.py: _f32_diags.npz'            ^ SyntaxError: invalid syntax 
65 0.296 examples/plot_shot.py         : OK!
66 0.154 examples/check_dw_pdt.py      : OK!
67 1.576 examples/wid_specgram.py      : a_data/', '/g/lhd_shots/'])<type 'exceptions.Exception'> 
68 8.008 examples/cluster_DA.py        : OK!
69 0.16 examples/check_dw_pdt1.py     : OK!
70 1.404 examples/igetfile.py          : OK!
71 0.306 examples/mode_identify_example_2012.py: orbar=colorbar NameError: name 'colorbar' is not defined 
72 2.358 examples/plot_signals_trivial.py: OK!
73 0.364 examples/small_65.py          : OK!
74 2.483 examples/ne_profile_preprocess.py: OK!
75 0.346 examples/small_300.py         : OK!
76 3.723 examples/fft_magnitude_extraction_LHD.py: OK!
77 0.491 examples/merge_basic_diagnostics.py: rror: [Errno 2] No such file or directory: 'DA_file.npz' 
78 0.285 examples/W7X_analysis.py      : le>     figure() NameError: name 'figure' is not defined 
79 3.604 examples/clean_up_pyfusion_text_mp.py: OK!
80 2.499 examples/plot_svd_old.py      : OK!
81 0.468 examples/merge_basic_HJ_diagnostics.py: sz = len(dd['shot']) NameError: name 'dd' is not defined 
82 0.833 examples/svd_simple_rotator.py: OK!
83 0.522 examples/test_check_buttons1.py: OK!
84 0.53 examples/testall.py           : OK!
85 0.501 examples/save_to_local.py     : OK!
86 0.938 examples/density_cluster.py   : OK!
87 0.461 examples/fonts.py             : OK!
88 0.012 examples/h1_mode_identify.py  : amples/cluster_info.py     ^ SyntaxError: invalid syntax 
89 1.583 examples/test_HJ.py           : OK!
90 0.024 examples/prepfs_range_sep.py  : utput_path prepfs_range_sep.py: error: too few arguments 
91 8.192 examples/restore_sin.py       : before DC adjust') NameError: name 'plot' is not defined 
92 2.733 examples/read_DF.py           : g/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags_comp5.h5...done 
93 0.482 examples/test_check_buttons.py: OK!
94 0.163 examples/__init__.py          : OK!
95 0.057 examples/modify_cfg.py        : OK!
96 0.164 examples/new_mode_identify_script_0.py: # need to adjust NameError: name 'arange' is not defined 
97 1.735 examples/channel_angle_games.py: OK!
98 2.801 examples/gen_fs_general.py    : OK!
99 0.105 examples/get_kappa.py         : >     kh = 0.*shot NameError: name 'shot' is not defined 
100 0.344 examples/flip_phase.py        : OK!
101 0.298 examples/plot_text_pyfusion.py: /pyfusion/examples/PF2_120227_HMPno12_27233_27233_1.dat' 
102 2.105 examples/check_polarity_flips.py: OK!
103 0.475 examples/play_with_fft_signals.py: OK!
104 0.138 examples/prepfs_range_mp.py   : OK!
55 good, 50 errors out of 105

This file was run on 28 Mar Feb 2016,  - future ones will be prepended.
 0 0.645 examples/plot_specgram.py     : ) ConfigParser.NoSectionError: No section: 'Diagnostic:' 
 1 0.493 examples/get_basic_params.py  : se ValueError('no good shots') ValueError: no good shots 
 2 1.401 examples/tables_extract.py    : OK!
 3 0.347 examples/slider_f_t.py        : ror: [Errno 2] No such file or directory: 'MP512all.npz' 
 4 2.51 examples/check_HD_histogram.py: OK!
 5 1.256 examples/prepfs_range_mp_2.py : OK!
 6 0.366 examples/merge_h1_diagnostics.py:  t_mid = dd['t_mid'] NameError: name 'dd' is not defined 
 7 0.511 examples/mode_identify_script_temp.py:  mode = mode_list[0] IndexError: list index out of range 
 8 0.593 examples/barber_pole.py       : OK!
 9 0.061 examples/example_fragments.py : overlap=220                ^ SyntaxError: invalid syntax 
10 0.493 examples/mode_identify_script.py:  mode = mode_list[0] IndexError: list index out of range 
11 0.188 examples/DA_troubles.py       : OK!
12 0.221 examples/setup_pyfusion.py    : OK!
13 0.312 examples/test_lasso_fs.py     : or key in dd.keys(): NameError: name 'dd' is not defined 
14 1.879 examples/medium_300.py        : OK!
15 0.057 examples/mode_identify_example_from_ICPP_2010.py: ll.txt' skip=4             ^ SyntaxError: invalid syntax 
16 7.476 examples/prepfs_range.py      : OK!
17 4.557 examples/gen_fs_local.py      : OK!
18 2.381 examples/plot_svd.py          : OK!
19 0.531 examples/process_swept_Langmuir_2.py:  the url is not available<type 'exceptions.LookupError'> 
20 0.845 examples/DP14_mode_plot.py    : OK!
21 0.056 examples/W7X_analysis_Fri_11Mar.py: sharey=2                   ^ SyntaxError: invalid syntax 
22 0.056 examples/compare_bands.py     : ption=()                   ^ SyntaxError: invalid syntax 
23 0.491 examples/mode_identify_script_old.py: ue') ValueError: Need to choose doN=True and/or doM=True 
24 0.505 examples/check_W7X_timebase.py:  the url is not available<type 'exceptions.LookupError'> 
25 0.52 examples/process_swept_Langmuir_1.py:  the url is not available<type 'exceptions.LookupError'> 
26 1.05 examples/plot_signals.py      : OK!
27 2.077 examples/trace_cluster.py     : kh','freq',ind=w23) NameError: name 'w23' is not defined 
28 0.435 examples/copy_DA.py           : ./datamining/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' 
29 0.057 examples/labelled_N_modes.py  : _list=None                 ^ SyntaxError: invalid syntax 
30 0.515 examples/process_swept_Langmuir.py:  the url is not available<type 'exceptions.LookupError'> 
31 6.866 examples/h1_mode_lists.py     : OK!
32 2.254 examples/cluster_info.py      : OK!
33 2.185 examples/gen_fs_bands.py      :  Time:  ########shot 27233: 0/0 fs finished in 0.30/1.38 
34 2.462 examples/phase_histogram.py   : OK!
35 0.495 examples/find_last_data.py    : .format(l=low))  ValueError: low limit of None not there 
36 1.15 examples/check_HMA_PMA_phase.py: OK!
37 2.288 examples/plot_svd1.py         : OK!
38 0.456 examples/find_close_shot.py   : OK!
39 0.526 examples/realtime_mode.py     : e also cm.gray_r etc NameError: name 'cm' is not defined 
40 0.33 examples/process_chirps.py    :  dfdtunit=dfdtunit)  NameError: name 'dd' is not defined 
41 1.625 examples/cross_phase.py       : OK!
42 0.313 examples/check_DA.py          : /datamining/old/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' 
43 0.098 examples/prepfs_time_mp.py    :  output_path prepfs_time_mp.py: error: too few arguments 
44 2.499 examples/plot_svd_simple.py   : OK!
45 0.307 examples/plot_many.py         : NOT AVAILABLE!!!!!') ValueError: SCAN NOT AVAILABLE!!!!! 
46 0.208 examples/query_h1ds.py        : soup import SqlSoup ImportError: No module named sqlsoup 
47 2.3 examples/plot_basic.py        : vide by zero encountered in true_divide   1/(dy1+dy2),)) 
48 0.497 examples/test_W7X_LP1.py      :  the url is not available<type 'exceptions.LookupError'> 
49 5.364 examples/cluster_phases_three_colour.py: to [10 11 12 13 14 15] - i.e. cldata has different sel,  
50 0.332 examples/new_mode_identify_script.py: ue') ValueError: Need to choose doN=True and/or doM=True 
51 3.499 examples/clean_up_pyfusion_text.py: OK!
52 0.347 examples/test_args.py         : OK!
53 0.769 examples/check_other_data.py  : OK!
54 0.059 examples/find_VSL_locking.py  : ist=mmm                    ^ SyntaxError: invalid syntax 
55 2.551 examples/cluster_DA_0.py      : OK!
56 0.307 examples/plot_text_fft.py     : r: [Errno 2] No such file or directory: 'g:fft26_50.dat' 
57 0.554 examples/merge_text_pyfusion.py: OK!
58 0.996 examples/multi_plot.py        : : igetfile None/wp/wp@105396.dat or .gz, .bz2 not found. 
59 2.042 examples/channel_spectra.py   : OK!
60 0.057 examples/shauns_clustering.py :         ^ SyntaxError: EOL while scanning string literal 
61 0.207 examples/plot_json_timeseries.py: OK!
62 0.329 examples/correct_VSL.py       : ppend_to_DA ImportError: cannot import name append_to_DA 
63 0.311 examples/fft_magnitude_extraction.py:  import HMA_funcs ImportError: No module named HMA_funcs 
64 0.056 examples/good_ones_density_cluster.py: _f32_diags.npz'            ^ SyntaxError: invalid syntax 
65 0.325 examples/plot_shot.py         : OK!
66 0.184 examples/check_dw_pdt.py      : OK!
67 1.445 examples/wid_specgram.py      : a_data/', '/g/lhd_shots/'])<type 'exceptions.Exception'> 
68 7.29 examples/cluster_DA.py        : OK!
69 0.191 examples/check_dw_pdt1.py     : OK!
70 1.179 examples/igetfile.py          : OK!
71 0.322 examples/mode_identify_example_2012.py: orbar=colorbar NameError: name 'colorbar' is not defined 
72 2.442 examples/plot_signals_trivial.py: OK!
73 0.388 examples/small_65.py          : OK!
74 2.734 examples/ne_profile_preprocess.py: OK!
75 0.367 examples/small_300.py         : OK!
76 3.586 examples/fft_magnitude_extraction_LHD.py: OK!
77 0.481 examples/merge_basic_diagnostics.py: rror: [Errno 2] No such file or directory: 'DA_file.npz' 
78 0.314 examples/W7X_analysis.py      : le>     figure() NameError: name 'figure' is not defined 
79 3.517 examples/clean_up_pyfusion_text_mp.py: OK!
80 2.267 examples/plot_svd_old.py      : OK!
81 0.499 examples/merge_basic_HJ_diagnostics.py: sz = len(dd['shot']) NameError: name 'dd' is not defined 
82 0.728 examples/svd_simple_rotator.py: OK!
83 0.483 examples/test_check_buttons1.py: OK!
84 0.522 examples/testall.py           : OK!
85 0.501 examples/save_to_local.py     : OK!
86 0.944 examples/density_cluster.py   : OK!
87 0.41 examples/fonts.py             : OK!
88 0.06 examples/h1_mode_identify.py  : amples/cluster_info.py     ^ SyntaxError: invalid syntax 
89 1.458 examples/test_HJ.py           : OK!
90 0.07 examples/prepfs_range_sep.py  : utput_path prepfs_range_sep.py: error: too few arguments 
91 8.054 examples/restore_sin.py       : st input array from shape (2,100000) into shape (100000) 
92 2.737 examples/read_DF.py           : g/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags_comp5.h5...done 
93 0.421 examples/test_check_buttons.py: OK!
94 0.192 examples/__init__.py          : OK!
95 0.19 examples/new_mode_identify_script_0.py: # need to adjust NameError: name 'arange' is not defined 
96 1.656 examples/channel_angle_games.py: OK!
97 2.269 examples/many_specs.py        : r import Node ImportError: No module named node_explorer 
98 2.668 examples/gen_fs_general.py    : OK!
99 0.128 examples/get_kappa.py         : >     kh = 0.*shot NameError: name 'shot' is not defined 
100 0.353 examples/flip_phase.py        : OK!
101 0.306 examples/plot_text_pyfusion.py: /pyfusion/examples/PF2_120227_HMPno12_27233_27233_1.dat' 
102 2.057 examples/check_polarity_flips.py: OK!
103 0.489 examples/play_with_fft_signals.py: OK!
104 0.172 examples/prepfs_range_mp.py   : OK!
51 errors out of 105  54 good

This file was run on 9 Mar Feb 2016, version 0.62a  - future ones will be prepended.
 0 0.45 examples/plot_specgram.py     : ) ConfigParser.NoSectionError: No section: 'Diagnostic:' 
 1 0.457 examples/get_basic_params.py  : se ValueError('no good shots') ValueError: no good shots 
 2 1.034 examples/tables_extract.py    : OK!
 3 0.274 examples/slider_f_t.py        : ror: [Errno 2] No such file or directory: 'MP512all.npz' 
 4 1.512 examples/check_HD_histogram.py: OK!
 5 1.206 examples/prepfs_range_mp_2.py : OK!
 6 0.335 examples/merge_h1_diagnostics.py:  t_mid = dd['t_mid'] NameError: name 'dd' is not defined 
 7 0.446 examples/mode_identify_script_temp.py:  mode = mode_list[0] IndexError: list index out of range 
 8 0.562 examples/barber_pole.py       : OK!
 9 0.017 examples/example_fragments.py : overlap=220                ^ SyntaxError: invalid syntax 
10 0.467 examples/mode_identify_script.py:  mode = mode_list[0] IndexError: list index out of range 
11 0.144 examples/DA_troubles.py       : OK!
12 0.174 examples/setup_pyfusion.py    : OK!
13 0.268 examples/test_lasso_fs.py     : or key in dd.keys(): NameError: name 'dd' is not defined 
14 1.497 examples/medium_300.py        : OK!
15 0.026 examples/mode_identify_example_from_ICPP_2010.py: ll.txt' skip=4             ^ SyntaxError: invalid syntax 
16 6.994 examples/prepfs_range.py      : OK!
17 4.208 examples/gen_fs_local.py      : OK!
18 2.22 examples/plot_svd.py          : OK!
19 0.812 examples/DP14_mode_plot.py    : OK!
20 0.023 examples/compare_bands.py     : ption=()                   ^ SyntaxError: invalid syntax 
21 0.459 examples/mode_identify_script_old.py: ue') ValueError: Need to choose doN=True and/or doM=True 
22 0.48 examples/check_W7X_timebase.py: 099900'><class 'future.backports.urllib.error.URLError'> 
23 0.989 examples/plot_signals.py      : OK!
24 2.003 examples/trace_cluster.py     : kh','freq',ind=w23) NameError: name 'w23' is not defined 
25 0.314 examples/copy_DA.py           : ./datamining/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' 
26 0.036 examples/labelled_N_modes.py  : _list=None                 ^ SyntaxError: invalid syntax 
27 6.813 examples/h1_mode_lists.py     : OK!
28 2.128 examples/cluster_info.py      : OK!
29 6.52 examples/gen_fs_bands.py      : :  ########shot 27233: 230/1428 fs finished in 4.88/5.83 
30 2.395 examples/phase_histogram.py   : OK!
31 0.457 examples/find_last_data.py    : .format(l=low))  ValueError: low limit of None not there 
32 1.039 examples/check_HMA_PMA_phase.py: OK!
33 2.194 examples/plot_svd1.py         : OK!
34 0.377 examples/find_close_shot.py   : OK!
35 0.509 examples/realtime_mode.py     : e also cm.gray_r etc NameError: name 'cm' is not defined 
36 0.263 examples/process_chirps.py    :  dfdtunit=dfdtunit)  NameError: name 'dd' is not defined 
37 1.496 examples/cross_phase.py       : OK!
38 0.286 examples/check_DA.py          : /datamining/old/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' 
39 0.066 examples/prepfs_time_mp.py    :  output_path prepfs_time_mp.py: error: too few arguments 
40 2.259 examples/plot_svd_simple.py   : OK!
41 0.278 examples/plot_many.py         : NOT AVAILABLE!!!!!') ValueError: SCAN NOT AVAILABLE!!!!! 
42 0.17 examples/query_h1ds.py        : soup import SqlSoup ImportError: No module named sqlsoup 
43 2.249 examples/plot_basic.py        : vide by zero encountered in true_divide   1/(dy1+dy2),)) 
44 0.478 examples/test_W7X_LP1.py      : 950061'><class 'future.backports.urllib.error.URLError'> 
45 5.149 examples/cluster_phases_three_colour.py: to [10 11 12 13 14 15] - i.e. cldata has different sel,  
46 0.31 examples/new_mode_identify_script.py: ue') ValueError: Need to choose doN=True and/or doM=True 
47 3.462 examples/clean_up_pyfusion_text.py: OK!
48 0.329 examples/test_args.py         : OK!
49 0.751 examples/check_other_data.py  : OK!
50 0.023 examples/find_VSL_locking.py  : ist=mmm                    ^ SyntaxError: invalid syntax 
51 2.776 examples/cluster_DA_0.py      : OK!
52 0.265 examples/plot_text_fft.py     : r: [Errno 2] No such file or directory: 'g:fft26_50.dat' 
53 0.534 examples/merge_text_pyfusion.py: OK!
54 0.951 examples/multi_plot.py        : : igetfile None/wp/wp@105396.dat or .gz, .bz2 not found. 
55 1.94 examples/channel_spectra.py   : OK!
56 0.018 examples/shauns_clustering.py :         ^ SyntaxError: EOL while scanning string literal 
57 0.179 examples/plot_json_timeseries.py: OK!
58 0.287 examples/correct_VSL.py       : ppend_to_DA ImportError: cannot import name append_to_DA 
59 0.275 examples/fft_magnitude_extraction.py:  import HMA_funcs ImportError: No module named HMA_funcs 
60 0.024 examples/good_ones_density_cluster.py: _f32_diags.npz'            ^ SyntaxError: invalid syntax 
61 0.277 examples/plot_shot.py         : OK!
62 0.153 examples/check_dw_pdt.py      : OK!
63 1.275 examples/wid_specgram.py      : l_data/', '/g/lhd_shots/'])<type 'exceptions.Exception'> 
64 7.498 examples/cluster_DA.py        : OK!
65 0.153 examples/check_dw_pdt1.py     : OK!
66 1.142 examples/igetfile.py          : OK!
67 0.285 examples/mode_identify_example_2012.py: orbar=colorbar NameError: name 'colorbar' is not defined 
68 2.155 examples/plot_signals_trivial.py: OK!
69 0.338 examples/small_65.py          : OK!
70 2.667 examples/ne_profile_preprocess.py: OK!
71 0.34 examples/small_300.py         : OK!
72 3.236 examples/fft_magnitude_extraction_LHD.py: OK!
73 0.442 examples/merge_basic_diagnostics.py: rror: [Errno 2] No such file or directory: 'DA_file.npz' 
74 3.398 examples/clean_up_pyfusion_text_mp.py: OK!
75 2.187 examples/plot_svd_old.py      : OK!
76 0.443 examples/merge_basic_HJ_diagnostics.py: sz = len(dd['shot']) NameError: name 'dd' is not defined 
77 0.723 examples/svd_simple_rotator.py: OK!
78 0.418 examples/test_check_buttons1.py: OK!
79 0.479 examples/testall.py           : OK!
80 0.929 examples/save_to_local.py     : OK!
81 0.878 examples/density_cluster.py   : OK!
82 0.367 examples/fonts.py             : OK!
83 0.018 examples/h1_mode_identify.py  : amples/cluster_info.py     ^ SyntaxError: invalid syntax 
84 1.382 examples/test_HJ.py           : OK!
85 0.029 examples/prepfs_range_sep.py  : utput_path prepfs_range_sep.py: error: too few arguments 
86 2.604 examples/read_DF.py           : g/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags_comp5.h5...done 
87 0.403 examples/test_check_buttons.py: OK!
88 0.163 examples/__init__.py          : OK!
89 0.171 examples/new_mode_identify_script_0.py: # need to adjust NameError: name 'arange' is not defined 
90 1.531 examples/channel_angle_games.py: OK!
91 2.192 examples/many_specs.py        : r import Node ImportError: No module named node_explorer 
92 2.679 examples/gen_fs_general.py    : OK!
93 0.109 examples/get_kappa.py         : >     kh = 0.*shot NameError: name 'shot' is not defined 
94 0.314 examples/flip_phase.py        : OK!
95 0.28 examples/plot_text_pyfusion.py: /pyfusion/examples/PF2_120227_HMPno12_27233_27233_1.dat' 
96 1.907 examples/check_polarity_flips.py: OK!
97 0.464 examples/play_with_fft_signals.py: OK!
98 0.145 examples/prepfs_range_mp.py   : OK!
45 errors out of 99 54 good  (actually only 43 errors when the W7 npz data is available)

This file was run on 21 Feb 2016, version 0.6  - future ones will be prepended.
==python 2.7.6==
 0 0.467 examples/plot_specgram.py     : ) ConfigParser.NoSectionError: No section: 'Diagnostic:' 
 1 0.475 examples/get_basic_params.py  : se ValueError('no good shots') ValueError: no good shots 
 2 1.103 examples/tables_extract.py    : OK!
 3 0.305 examples/slider_f_t.py        : ror: [Errno 2] No such file or directory: 'MP512all.npz' 
 4 1.686 examples/check_HD_histogram.py: OK!
 5 1.284 examples/prepfs_range_mp_2.py : OK!
 6 0.331 examples/merge_h1_diagnostics.py:  t_mid = dd['t_mid'] NameError: name 'dd' is not defined 
 7 0.491 examples/mode_identify_script_temp.py:  mode = mode_list[0] IndexError: list index out of range 
 8 0.582 examples/barber_pole.py       : OK!
 9 0.013 examples/example_fragments.py : overlap=220                ^ SyntaxError: invalid syntax 
10 0.464 examples/mode_identify_script.py:  mode = mode_list[0] IndexError: list index out of range 
11 0.168 examples/DA_troubles.py       : OK!
12 0.175 examples/setup_pyfusion.py    : OK!
13 0.266 examples/test_lasso_fs.py     : or key in dd.keys(): NameError: name 'dd' is not defined 
14 1.657 examples/medium_300.py        : OK!
15 0.016 examples/mode_identify_example_from_ICPP_2010.py: ll.txt' skip=4             ^ SyntaxError: invalid syntax 
16 7.659 examples/prepfs_range.py      : OK!
17 4.626 examples/gen_fs_local.py      : OK!
18 2.497 examples/plot_svd.py          : OK!
19 0.839 examples/DP14_mode_plot.py    : OK!
20 0.012 examples/compare_bands.py     : ption=()                   ^ SyntaxError: invalid syntax 
21 0.464 examples/mode_identify_script_old.py: ue') ValueError: Need to choose doN=True and/or doM=True 
22 1.072 examples/plot_signals.py      : OK!
23 2.177 examples/trace_cluster.py     : kh','freq',ind=w23) NameError: name 'w23' is not defined 
24 0.366 examples/copy_DA.py           : ./datamining/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' 
25 0.016 examples/labelled_N_modes.py  : _list=None                 ^ SyntaxError: invalid syntax 
26 7.272 examples/h1_mode_lists.py     : OK!
27 2.499 examples/cluster_info.py      : OK!
28 7.045 examples/gen_fs_bands.py      : :  ########shot 27233: 230/1428 fs finished in 5.33/6.38 
29 2.593 examples/phase_histogram.py   : OK!
30 0.454 examples/find_last_data.py    : .format(l=low))  ValueError: low limit of None not there 
31 1.12 examples/check_HMA_PMA_phase.py: OK!
32 2.424 examples/plot_svd1.py         : OK!
33 0.459 examples/find_close_shot.py   : OK!
34 0.521 examples/realtime_mode.py     : e also cm.gray_r etc NameError: name 'cm' is not defined 
35 0.269 examples/process_chirps.py    :  dfdtunit=dfdtunit)  NameError: name 'dd' is not defined 
36 1.687 examples/cross_phase.py       : OK!
37 0.267 examples/check_DA.py          : /datamining/old/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' 
38 0.072 examples/prepfs_time_mp.py    :  output_path prepfs_time_mp.py: error: too few arguments 
39 2.468 examples/plot_svd_simple.py   : OK!
40 0.256 examples/plot_many.py         : NOT AVAILABLE!!!!!') ValueError: SCAN NOT AVAILABLE!!!!! 
41 0.196 examples/query_h1ds.py        : soup import SqlSoup ImportError: No module named sqlsoup 
42 2.551 examples/plot_basic.py        : vide by zero encountered in true_divide   1/(dy1+dy2),)) 
43 5.692 examples/cluster_phases_three_colour.py: to [10 11 12 13 14 15] - i.e. cldata has different sel,  
44 0.286 examples/new_mode_identify_script.py: ue') ValueError: Need to choose doN=True and/or doM=True 
45 3.604 examples/clean_up_pyfusion_text.py: OK!
46 0.318 examples/test_args.py         : OK!
47 7.085 examples/check_other_data.py  : OK!
48 0.017 examples/find_VSL_locking.py  : ist=mmm                    ^ SyntaxError: invalid syntax 
49 2.872 examples/cluster_DA_0.py      : OK!
50 0.256 examples/plot_text_fft.py     : r: [Errno 2] No such file or directory: 'g:fft26_50.dat' 
51 0.553 examples/merge_text_pyfusion.py: OK!
52 1.013 examples/multi_plot.py        : : igetfile None/wp/wp@105396.dat or .gz, .bz2 not found. 
53 2.149 examples/channel_spectra.py   : OK!
54 0.019 examples/shauns_clustering.py :         ^ SyntaxError: EOL while scanning string literal 
55 0.276 examples/correct_VSL.py       : ppend_to_DA ImportError: cannot import name append_to_DA 
56 0.269 examples/fft_magnitude_extraction.py:  import HMA_funcs ImportError: No module named HMA_funcs 
57 0.013 examples/good_ones_density_cluster.py: _f32_diags.npz'            ^ SyntaxError: invalid syntax 
58 0.28 examples/plot_shot.py         : OK!
59 0.165 examples/check_dw_pdt.py      : OK!
60 1.393 examples/wid_specgram.py      : l_data/', '/g/lhd_shots/'])<type 'exceptions.Exception'> 
61 8.149 examples/cluster_DA.py        : OK!
62 0.17 examples/check_dw_pdt1.py     : OK!
63 1.208 examples/igetfile.py          : OK!
64 0.27 examples/mode_identify_example_2012.py: orbar=colorbar NameError: name 'colorbar' is not defined 
65 2.533 examples/plot_signals_trivial.py: OK!
66 0.341 examples/small_65.py          : OK!
67 3.21 examples/ne_profile_preprocess.py: OK!
68 0.313 examples/small_300.py         : OK!
69 3.615 examples/fft_magnitude_extraction_LHD.py: OK!
70 0.46 examples/merge_basic_diagnostics.py: rror: [Errno 2] No such file or directory: 'DA_file.npz' 
71 3.681 examples/clean_up_pyfusion_text_mp.py: OK!
72 2.463 examples/plot_svd_old.py      : OK!
73 0.451 examples/merge_basic_HJ_diagnostics.py: sz = len(dd['shot']) NameError: name 'dd' is not defined 
74 0.777 examples/svd_simple_rotator.py: OK!
75 0.475 examples/test_check_buttons1.py: OK!
76 0.49 examples/testall.py           : OK!
77 1.003 examples/save_to_local.py     : OK!
78 7.755 examples/density_cluster.py   : OK!
79 0.414 examples/fonts.py             : OK!
80 0.013 examples/h1_mode_identify.py  : amples/cluster_info.py     ^ SyntaxError: invalid syntax 
81 1.491 examples/test_HJ.py           : OK!
82 0.024 examples/prepfs_range_sep.py  : utput_path prepfs_range_sep.py: error: too few arguments 
83 2.868 examples/read_DF.py           : g/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags_comp5.h5...done 
84 0.44 examples/test_check_buttons.py: OK!
85 0.162 examples/__init__.py          : OK!
86 0.174 examples/new_mode_identify_script_0.py: # need to adjust NameError: name 'arange' is not defined 
87 1.711 examples/channel_angle_games.py: OK!
88 2.347 examples/many_specs.py        : r import Node ImportError: No module named node_explorer 
89 2.88 examples/gen_fs_general.py    : OK!
90 0.108 examples/get_kappa.py         : >     kh = 0.*shot NameError: name 'shot' is not defined 
91 0.31 examples/flip_phase.py        : OK!
92 0.287 examples/plot_text_pyfusion.py: /pyfusion/examples/PF2_120227_HMPno12_27233_27233_1.dat' 
93 2.089 examples/check_polarity_flips.py: OK!
94 0.482 examples/play_with_fft_signals.py: OK!
95 0.14 examples/prepfs_range_mp.py   : OK!
43 errors out of 96: 53 good

== Python 3.5.1 21 Feb 2016 - no improvement,,bt also no new bugs
 0 0.878 examples/plot_specgram.py     : b") configparser.NoSectionError: No section: 'Diagnostic:' "
 1 0.932 examples/get_basic_params.py  : b"se ValueError('no good shots') ValueError: no good shots "
 2 1.483 examples/tables_extract.py    : b'OK!'
 3 0.483 examples/slider_f_t.py        : b"ror: [Errno 2] No such file or directory: 'MP512all.npz' "
 4 1.857 examples/check_HD_histogram.py: b'OK!'
 5 2.208 examples/prepfs_range_mp_2.py : b'OK!'
 6 0.256 examples/merge_h1_diagnostics.py: b"t__(name, gbls) ImportError: No module named 'mdsdevice' "
 7 0.874 examples/mode_identify_script_temp.py: b'e") LookupError: dd not loaded into memory - can\'t store '
 8 0.685 examples/barber_pole.py       : b'OK!'
 9 0.021 examples/example_fragments.py : b'overlap=220                ^ SyntaxError: invalid syntax '
10 0.815 examples/mode_identify_script.py: b' mode = mode_list[0] IndexError: list index out of range '
11 0.391 examples/DA_troubles.py       : b'OK!'
12 0.301 examples/setup_pyfusion.py    : b"mport.install() ImportError: No module named 'pyximport' "
13 0.527 examples/test_lasso_fs.py     : b"or key in dd.keys(): NameError: name 'dd' is not defined "
14 1.92 examples/medium_300.py        : b'OK!'
15 0.031 examples/mode_identify_example_from_ICPP_2010.py: b"ll.txt' skip=4             ^ SyntaxError: invalid syntax "
16 15.854 examples/prepfs_range.py      : b'OK!'
17 8.915 examples/gen_fs_local.py      : b'OK!'
18 1.26 examples/plot_svd.py          : b"q::top.acq216_026:input_02 name 'MDSplus' is not defined "
19 0.954 examples/DP14_mode_plot.py    : b'OK!'
20 0.031 examples/compare_bands.py     : b'ption=()                   ^ SyntaxError: invalid syntax '
21 0.906 examples/mode_identify_script_old.py: b'e") LookupError: dd not loaded into memory - can\'t store '
22 1.126 examples/plot_signals.py      : b'OK!'
23 0.784 examples/trace_cluster.py     : b"ile(filename,'r')) NameError: name 'file' is not defined "
24 0.731 examples/copy_DA.py           : b"./datamining/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' "
25 0.032 examples/labelled_N_modes.py  : b'_list=None                 ^ SyntaxError: invalid syntax '
26 0.767 examples/h1_mode_lists.py     : b"ile(filename,'r')) NameError: name 'file' is not defined "
27 0.781 examples/cluster_info.py      : b"ile(filename,'r')) NameError: name 'file' is not defined "
28 13.161 examples/gen_fs_bands.py      : b' ########shot 27233: 230/1428 fs finished in 11.06/11.97 '
29 2.812 examples/phase_histogram.py   : b'OK!'
30 1.106 examples/find_last_data.py    : b'.format(l=low))  ValueError: low limit of None not there '
31 1.082 examples/check_HMA_PMA_phase.py: b"nov::top.ACQ132_7:input_27 name 'MDSplus' is not defined "
32 0.032 examples/plot_svd1.py         : b'                           ^ SyntaxError: invalid syntax '
33 0.686 examples/find_close_shot.py   : b' You may need to pass the encoding= option to numpy.load '
34 0.656 examples/realtime_mode.py     : b"t__(name, gbls) ImportError: No module named 'mdsdevice' "
35 0.525 examples/process_chirps.py    : b" dfdtunit=dfdtunit)  NameError: name 'dd' is not defined "
36 1.008 examples/cross_phase.py       : b"nov::top.ACQ132_8:input_05 name 'MDSplus' is not defined "
37 0.591 examples/check_DA.py          : b"/datamining/old/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' "
38 0.16 examples/prepfs_time_mp.py    : b'error: the following arguments are required: output_path '
39 0.992 examples/plot_svd_simple.py   : b"q::top.acq216_026:input_02 name 'MDSplus' is not defined "
40 0.031 examples/plot_many.py         : b'(),shot_list))             ^ SyntaxError: invalid syntax '
41 0.441 examples/query_h1ds.py        : b"up ImportError: No module named 'sqlalchemy.ext.sqlsoup' "
42 1.024 examples/plot_basic.py        : b'alid mode: %r" % (mode,)) ValueError: Invalid mode: \'rU\' '
43 0.543 examples/cluster_phases_three_colour.py: b"wer() != 'n': NameError: name 'raw_input' is not defined "
44 0.385 examples/new_mode_identify_script.py: b"esis.pickle','r')) NameError: name 'file' is not defined "
45 10.178 examples/clean_up_pyfusion_text.py: b'OK!'
46 0.555 examples/test_args.py         : b'OK!'
47 0.025 examples/check_other_data.py  : b'rror: inconsistent use of tabs and spaces in indentation '
48 0.025 examples/find_VSL_locking.py  : b'ist=mmm                    ^ SyntaxError: invalid syntax '
49 3.744 examples/cluster_DA_0.py      : b'OK!'
50 0.556 examples/plot_text_fft.py     : b"r: [Errno 2] No such file or directory: 'g:fft26_50.dat' "
51 1.251 examples/merge_text_pyfusion.py: b'OK!'
52 1.23 examples/multi_plot.py        : b': igetfile None/wp/wp@105396.dat or .gz, .bz2 not found. '
53 0.998 examples/channel_spectra.py   : b"nov::top.ACQ132_7:input_18 name 'MDSplus' is not defined "
54 0.031 examples/shauns_clustering.py : b'        ^ SyntaxError: EOL while scanning string literal '
55 0.543 examples/correct_VSL.py       : b"end_to_DA ImportError: cannot import name 'append_to_DA' "
56 0.031 examples/fft_magnitude_extraction.py: b"   ^ SyntaxError: Missing parentheses in call to 'print' "
57 0.023 examples/good_ones_density_cluster.py: b"_f32_diags.npz'            ^ SyntaxError: invalid syntax "
58 0.528 examples/plot_shot.py         : b'OK!'
59 0.38 examples/check_dw_pdt.py      : b'OK!'
60 0.028 examples/wid_specgram.py      : b'p, _type, fmod             ^ SyntaxError: invalid syntax '
61 8.079 examples/cluster_DA.py        : b'rnings.warn("Setting the \'color\' property will override" '
62 0.408 examples/check_dw_pdt1.py     : b'OK!'
63 1.429 examples/igetfile.py          : b'OK!'
64 0.55 examples/mode_identify_example_2012.py: b"orbar=colorbar NameError: name 'colorbar' is not defined "
65 1.983 examples/plot_signals_trivial.py: b'OK!'
66 0.645 examples/small_65.py          : b'OK!'
67 0.035 examples/ne_profile_preprocess.py: b'                           ^ SyntaxError: invalid syntax '
68 0.575 examples/small_300.py         : b'OK!'
69 0.026 examples/fft_magnitude_extraction_LHD.py: b"   ^ SyntaxError: Missing parentheses in call to 'print' "
70 0.893 examples/merge_basic_diagnostics.py: b"rror: [Errno 2] No such file or directory: 'DA_file.npz' "
71 11.916 examples/clean_up_pyfusion_text_mp.py: b'OK!'
72 0.022 examples/plot_svd_old.py      : b'                           ^ SyntaxError: invalid syntax '
73 0.999 examples/merge_basic_HJ_diagnostics.py: b"sz = len(dd['shot']) NameError: name 'dd' is not defined "
74 0.024 examples/svd_simple_rotator.py: b'hronof)))                  ^ SyntaxError: invalid syntax '
75 0.628 examples/test_check_buttons1.py: b'OK!'
76 1.124 examples/testall.py           : b"wer() != 'n': NameError: name 'raw_input' is not defined "
77 1.136 examples/save_to_local.py     : b") TypeError: type Signal doesn't define __round__ method "
78 0.642 examples/density_cluster.py   : b"AttributeError: 'range' object has no attribute 'extend' "
79 0.621 examples/fonts.py             : b'OK!'
80 0.029 examples/h1_mode_identify.py  : b'amples/cluster_info.py     ^ SyntaxError: invalid syntax '
81 1.872 examples/test_HJ.py           : b'OK!'
82 0.058 examples/prepfs_range_sep.py  : b'error: the following arguments are required: output_path '
83 3.93 examples/read_DF.py           : b'g/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags_comp5.h5...done '
84 0.623 examples/test_check_buttons.py: b'OK!'
85 0.408 examples/__init__.py          : b'OK!'
86 0.394 examples/new_mode_identify_script_0.py: b"# need to adjust NameError: name 'arange' is not defined "
87 1.814 examples/channel_angle_games.py: b'OK!'
88 0.031 examples/many_specs.py        : b"   ^ SyntaxError: Missing parentheses in call to 'print' "
89 4.49 examples/gen_fs_general.py    : b'n.MDSPlus.fetch import MDSPlusDataFetcher, get_tree_path '
90 0.222 examples/get_kappa.py         : b"t__(name, gbls) ImportError: No module named 'mdsdevice' "
91 0.572 examples/flip_phase.py        : b'OK!'
92 0.036 examples/plot_text_pyfusion.py: b'                           ^ SyntaxError: invalid syntax '
93 2.013 examples/check_polarity_flips.py: b'OK!'
94 0.922 examples/play_with_fft_signals.py: b'OK!'
95 0.187 examples/prepfs_range_mp.py   : b"ue TypeError: a bytes-like object is required, not 'str' "
64 errors out of 96 - 32 good.

This file was run on 7 Feb 2016, - future ones will be prepended.
==python 2.7.6==
 0 3.064 examples/plot_specgram.py     : t_axis_if_OK(pl.gca(),axt[0:2], np.array(axt[2:])/ffact) 
 1 0.416 examples/get_basic_params.py  : se ValueError('no good shots') ValueError: no good shots 
 2 1.358 examples/tables_extract.py    : OK!
 3 0.331 examples/slider_f_t.py        : ror: [Errno 2] No such file or directory: 'MP512all.npz' 
 4 3.044 examples/check_HD_histogram.py: OK!
 5 1.168 examples/prepfs_range_mp_2.py : OK!
 6 0.294 examples/merge_h1_diagnostics.py:  t_mid = dd['t_mid'] NameError: name 'dd' is not defined 
 7 0.465 examples/mode_identify_script_temp.py:  mode = mode_list[0] IndexError: list index out of range 
 8 0.513 examples/barber_pole.py       : OK!
 9 0.014 examples/example_fragments.py : overlap=220                ^ SyntaxError: invalid syntax 
10 0.448 examples/mode_identify_script.py:  mode = mode_list[0] IndexError: list index out of range 
11 0.147 examples/DA_troubles.py       : OK!
12 0.153 examples/setup_pyfusion.py    : OK!
13 0.242 examples/test_lasso_fs.py     : or key in dd.keys(): NameError: name 'dd' is not defined 
14 1.868 examples/medium_300.py        : OK!
15 0.012 examples/mode_identify_example_from_ICPP_2010.py: ll.txt' skip=4             ^ SyntaxError: invalid syntax 
16 8.331 examples/prepfs_range.py      : OK!
17 5.159 examples/gen_fs_local.py      : OK!
18 3.388 examples/plot_svd.py          : OK!
19 0.952 examples/DP14_mode_plot.py    : OK!
20 0.488 examples/mode_identify_script_old.py: ue') ValueError: Need to choose doN=True and/or doM=True 
21 1.342 examples/plot_signals.py      : OK!
22 2.578 examples/trace_cluster.py     : kh','freq',ind=w23) NameError: name 'w23' is not defined 
23 0.46 examples/copy_DA.py           : ./datamining/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' 
24 0.012 examples/labelled_N_modes.py  : _list=None                 ^ SyntaxError: invalid syntax 
25 8.001 examples/h1_mode_lists.py     : OK!
26 2.742 examples/cluster_info.py      : OK!
27 7.618 examples/gen_fs_bands.py      : :  ########shot 27233: 230/1435 fs finished in 5.40/6.92 
28 2.558 examples/phase_histogram.py   : OK!
29 0.421 examples/find_last_data.py    : .format(l=low))  ValueError: low limit of None not there 
30 1.093 examples/check_HMA_PMA_phase.py: OK!
31 2.884 examples/plot_svd1.py         : OK!
32 0.436 examples/find_close_shot.py   : OK!
33 0.445 examples/realtime_mode.py     : e also cm.gray_r etc NameError: name 'cm' is not defined 
34 0.247 examples/process_chirps.py    :  dfdtunit=dfdtunit)  NameError: name 'dd' is not defined 
35 1.593 examples/cross_phase.py       : OK!
36 0.263 examples/check_DA.py          : /datamining/old/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' 
37 0.064 examples/prepfs_time_mp.py    :  output_path prepfs_time_mp.py: error: too few arguments 
38 2.46 examples/plot_svd_simple.py   : OK!
39 0.261 examples/plot_many.py         : NOT AVAILABLE!!!!!') ValueError: SCAN NOT AVAILABLE!!!!! 
40 0.17 examples/query_h1ds.py        : soup import SqlSoup ImportError: No module named sqlsoup 
41 2.382 examples/plot_basic.py        : vide by zero encountered in true_divide   1/(dy1+dy2),)) 
42 5.771 examples/cluster_phases_three_colour.py: to [10 11 12 13 14 15] - i.e. cldata has different sel,  
43 0.276 examples/new_mode_identify_script.py: ue') ValueError: Need to choose doN=True and/or doM=True 
44 3.655 examples/clean_up_pyfusion_text.py: OK!
45 0.289 examples/test_args.py         : OK!
46 10.779 examples/check_other_data.py  : OK!
47 0.013 examples/find_VSL_locking.py  : ist=mmm                    ^ SyntaxError: invalid syntax 
48 2.969 examples/cluster_DA_0.py      : OK!
49 0.249 examples/plot_text_fft.py     : r: [Errno 2] No such file or directory: 'g:fft26_50.dat' 
50 0.49 examples/merge_text_pyfusion.py: OK!
51 0.959 examples/multi_plot.py        : : igetfile None/wp/wp@105396.dat or .gz, .bz2 not found. 
52 0.013 examples/shauns_clustering.py :         ^ SyntaxError: EOL while scanning string literal 
53 0.257 examples/correct_VSL.py       : ppend_to_DA ImportError: cannot import name append_to_DA 
54 0.248 examples/fft_magnitude_extraction.py:  import HMA_funcs ImportError: No module named HMA_funcs 
55 0.013 examples/good_ones_density_cluster.py: _f32_diags.npz'            ^ SyntaxError: invalid syntax 
56 0.249 examples/plot_shot.py         : OK!
57 0.151 examples/check_dw_pdt.py      : OK!
58 1.441 examples/wid_specgram.py      : l_data/', '/g/lhd_shots/'])<type 'exceptions.Exception'> 
59 8.181 examples/cluster_DA.py        : OK!
60 0.147 examples/check_dw_pdt1.py     : OK!
61 1.223 examples/igetfile.py          : OK!
62 0.272 examples/mode_identify_example_2012.py: orbar=colorbar NameError: name 'colorbar' is not defined 
63 3.091 examples/plot_signals_trivial.py: OK!
64 0.336 examples/small_65.py          : OK!
65 3.138 examples/ne_profile_preprocess.py: OK!
66 0.303 examples/small_300.py         : OK!
67 4.197 examples/fft_magnitude_extraction_LHD.py: OK!
68 0.438 examples/merge_basic_diagnostics.py: rror: [Errno 2] No such file or directory: 'DA_file.npz' 
69 3.67 examples/clean_up_pyfusion_text_mp.py: OK!
70 2.906 examples/plot_svd_old.py      : OK!
71 0.282 examples/merge_basic_HJ_diagnostics.py: f ierror != 0:     ^ IndentationError: unexpected indent 
72 0.768 examples/svd_simple_rotator.py: OK!
73 0.504 examples/test_check_buttons1.py: OK!
74 0.479 examples/testall.py           : OK!
75 1.391 examples/save_to_local.py     : OK!
76 9.554 examples/density_cluster.py   : OK!
77 0.404 examples/fonts.py             : OK!
78 0.018 examples/h1_mode_identify.py  : amples/cluster_info.py     ^ SyntaxError: invalid syntax 
79 1.397 examples/test_HJ.py           : a not found for 50136:MP1<type 'exceptions.LookupError'> 
80 0.023 examples/prepfs_range_sep.py  : utput_path prepfs_range_sep.py: error: too few arguments 
81 2.935 examples/read_DF.py           : g/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags_comp5.h5...done 
82 0.407 examples/test_check_buttons.py: OK!
83 0.158 examples/__init__.py          : OK!
84 0.155 examples/new_mode_identify_script_0.py: # need to adjust NameError: name 'arange' is not defined 
85 2.462 examples/channel_angle_games.py: OK!
86 2.653 examples/many_specs.py        : r import Node ImportError: No module named node_explorer 
87 2.838 examples/gen_fs_general.py    : OK!
88 0.091 examples/get_kappa.py         : >     kh = 0.*shot NameError: name 'shot' is not defined 
89 0.287 examples/flip_phase.py        : OK!
90 0.251 examples/plot_text_pyfusion.py: /pyfusion/examples/PF2_120227_HMPno12_27233_27233_1.dat' 
91 2.804 examples/check_polarity_flips.py: OK!
92 0.424 examples/play_with_fft_signals.py: OK!
93 0.131 examples/prepfs_range_mp.py   : OK!
43 errors out of 94: 51 good

== Python 3.5 7 Feb 2016 - no improvement,,bt also no new bugs
 0 2.813 examples/plot_specgram.py     : b'OK!'
 1 0.964 examples/get_basic_params.py  : b"se ValueError('no good shots') ValueError: no good shots "
 2 1.463 examples/tables_extract.py    : b'OK!'
 3 0.444 examples/slider_f_t.py        : b"ror: [Errno 2] No such file or directory: 'MP512all.npz' "
 4 3.158 examples/check_HD_histogram.py: b'OK!'
 5 2.034 examples/prepfs_range_mp_2.py : b'OK!'
 6 0.231 examples/merge_h1_diagnostics.py: b"t__(name, gbls) ImportError: No module named 'mdsdevice' "
 7 0.78 examples/mode_identify_script_temp.py: b'e") LookupError: dd not loaded into memory - can\'t store '
 8 0.658 examples/barber_pole.py       : b'OK!'
 9 0.032 examples/example_fragments.py : b'overlap=220                ^ SyntaxError: invalid syntax '
10 0.751 examples/mode_identify_script.py: b' mode = mode_list[0] IndexError: list index out of range '
11 0.341 examples/DA_troubles.py       : b'OK!'
12 0.288 examples/setup_pyfusion.py    : b"mport.install() ImportError: No module named 'pyximport' "
13 0.509 examples/test_lasso_fs.py     : b"or key in dd.keys(): NameError: name 'dd' is not defined "
14 2.044 examples/medium_300.py        : b'OK!'
15 0.024 examples/mode_identify_example_from_ICPP_2010.py: b"ll.txt' skip=4             ^ SyntaxError: invalid syntax "
16 15.448 examples/prepfs_range.py      : b'OK!'
17 8.233 examples/gen_fs_local.py      : b'OK!'
18 1.063 examples/plot_svd.py          : b"q::top.acq216_026:input_02 name 'MDSplus' is not defined "
19 0.917 examples/DP14_mode_plot.py    : b'OK!'
20 0.809 examples/mode_identify_script_old.py: b'e") LookupError: dd not loaded into memory - can\'t store '
21 1.037 examples/plot_signals.py      : b'OK!'
22 0.804 examples/trace_cluster.py     : b"ile(filename,'r')) NameError: name 'file' is not defined "
23 0.684 examples/copy_DA.py           : b"./datamining/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' "
24 0.027 examples/labelled_N_modes.py  : b'_list=None                 ^ SyntaxError: invalid syntax '
25 0.811 examples/h1_mode_lists.py     : b"ile(filename,'r')) NameError: name 'file' is not defined "
26 0.752 examples/cluster_info.py      : b"ile(filename,'r')) NameError: name 'file' is not defined "
27 14.772 examples/gen_fs_bands.py      : b' ########shot 27233: 230/1435 fs finished in 12.26/13.66 '
28 2.404 examples/phase_histogram.py   : b'OK!'
29 0.812 examples/find_last_data.py    : b'.format(l=low))  ValueError: low limit of None not there '
30 0.907 examples/check_HMA_PMA_phase.py: b"nov::top.ACQ132_7:input_27 name 'MDSplus' is not defined "
31 0.027 examples/plot_svd1.py         : b'                           ^ SyntaxError: invalid syntax '
32 0.682 examples/find_close_shot.py   : b' You may need to pass the encoding= option to numpy.load '
33 0.636 examples/realtime_mode.py     : b"t__(name, gbls) ImportError: No module named 'mdsdevice' "
34 0.644 examples/process_chirps.py    : b" dfdtunit=dfdtunit)  NameError: name 'dd' is not defined "
35 1.102 examples/cross_phase.py       : b"nov::top.ACQ132_8:input_05 name 'MDSplus' is not defined "
36 0.542 examples/check_DA.py          : b"/datamining/old/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' "
37 0.16 examples/prepfs_time_mp.py    : b'error: the following arguments are required: output_path '
38 0.926 examples/plot_svd_simple.py   : b"q::top.acq216_026:input_02 name 'MDSplus' is not defined "
39 0.028 examples/plot_many.py         : b'(),shot_list))             ^ SyntaxError: invalid syntax '
40 0.469 examples/query_h1ds.py        : b"up ImportError: No module named 'sqlalchemy.ext.sqlsoup' "
41 1.077 examples/plot_basic.py        : b'alid mode: %r" % (mode,)) ValueError: Invalid mode: \'rU\' '
42 0.548 examples/cluster_phases_three_colour.py: b"wer() != 'n': NameError: name 'raw_input' is not defined "
43 0.406 examples/new_mode_identify_script.py: b"esis.pickle','r')) NameError: name 'file' is not defined "
44 16.311 examples/clean_up_pyfusion_text.py: b'OK!'
45 0.709 examples/test_args.py         : b'OK!'
46 0.026 examples/check_other_data.py  : b'rror: inconsistent use of tabs and spaces in indentation '
47 0.027 examples/find_VSL_locking.py  : b'ist=mmm                    ^ SyntaxError: invalid syntax '
48 3.511 examples/cluster_DA_0.py      : b'OK!'
49 0.531 examples/plot_text_fft.py     : b"r: [Errno 2] No such file or directory: 'g:fft26_50.dat' "
50 1.505 examples/merge_text_pyfusion.py: b'OK!'
51 1.089 examples/multi_plot.py        : b': igetfile None/wp/wp@105396.dat or .gz, .bz2 not found. '
52 0.045 examples/shauns_clustering.py : b'        ^ SyntaxError: EOL while scanning string literal '
53 0.551 examples/correct_VSL.py       : b"end_to_DA ImportError: cannot import name 'append_to_DA' "
54 0.027 examples/fft_magnitude_extraction.py: b"   ^ SyntaxError: Missing parentheses in call to 'print' "
55 0.026 examples/good_ones_density_cluster.py: b"_f32_diags.npz'            ^ SyntaxError: invalid syntax "
56 0.54 examples/plot_shot.py         : b'OK!'
57 0.412 examples/check_dw_pdt.py      : b'OK!'
58 0.031 examples/wid_specgram.py      : b'p, _type, fmod             ^ SyntaxError: invalid syntax '
59 8.851 examples/cluster_DA.py        : b'rnings.warn("Setting the \'color\' property will override" '
60 0.411 examples/check_dw_pdt1.py     : b'OK!'
61 1.319 examples/igetfile.py          : b'OK!'
62 0.722 examples/mode_identify_example_2012.py: b"orbar=colorbar NameError: name 'colorbar' is not defined "
63 2.299 examples/plot_signals_trivial.py: b'OK!'
64 0.644 examples/small_65.py          : b'OK!'
65 0.028 examples/ne_profile_preprocess.py: b'                           ^ SyntaxError: invalid syntax '
66 0.601 examples/small_300.py         : b'OK!'
67 0.028 examples/fft_magnitude_extraction_LHD.py: b"   ^ SyntaxError: Missing parentheses in call to 'print' "
68 0.847 examples/merge_basic_diagnostics.py: b"rror: [Errno 2] No such file or directory: 'DA_file.npz' "
69 10.685 examples/clean_up_pyfusion_text_mp.py: b'OK!'
70 0.029 examples/plot_svd_old.py      : b'                           ^ SyntaxError: invalid syntax '
71 0.55 examples/merge_basic_HJ_diagnostics.py: b'f ierror != 0:     ^ IndentationError: unexpected indent '
72 0.028 examples/svd_simple_rotator.py: b'hronof)))                  ^ SyntaxError: invalid syntax '
73 0.585 examples/test_check_buttons1.py: b'OK!'
74 1.236 examples/testall.py           : b"wer() != 'n': NameError: name 'raw_input' is not defined "
75 1.284 examples/save_to_local.py     : b") TypeError: type Signal doesn't define __round__ method "
76 0.607 examples/density_cluster.py   : b"AttributeError: 'range' object has no attribute 'extend' "
77 0.554 examples/fonts.py             : b'OK!'
78 0.04 examples/h1_mode_identify.py  : b'amples/cluster_info.py     ^ SyntaxError: invalid syntax '
79 1.676 examples/test_HJ.py           : b" style data not found for 50136:MP1<class 'LookupError'> "
80 0.059 examples/prepfs_range_sep.py  : b'error: the following arguments are required: output_path '
81 3.85 examples/read_DF.py           : b'g/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags_comp5.h5...done '
82 0.572 examples/test_check_buttons.py: b'OK!'
83 0.439 examples/__init__.py          : b'OK!'
84 0.412 examples/new_mode_identify_script_0.py: b"# need to adjust NameError: name 'arange' is not defined "
85 1.866 examples/channel_angle_games.py: b'OK!'
86 0.04 examples/many_specs.py        : b"   ^ SyntaxError: Missing parentheses in call to 'print' "
87 3.983 examples/gen_fs_general.py    : b'n.MDSPlus.fetch import MDSPlusDataFetcher, get_tree_path '
88 0.246 examples/get_kappa.py         : b"t__(name, gbls) ImportError: No module named 'mdsdevice' "
89 0.589 examples/flip_phase.py        : b'OK!'
90 0.027 examples/plot_text_pyfusion.py: b'                           ^ SyntaxError: invalid syntax '
91 1.898 examples/check_polarity_flips.py: b'OK!'
92 0.806 examples/play_with_fft_signals.py: b'OK!'
93 0.172 examples/prepfs_range_mp.py   : b"tributeError: module 'itertools' has no attribute 'izip' "
62 errors out of 94 - 32 good


This file was run on 30 Dec 2015, - future ones will be prepended.
==python 2.7.6==
 0 2.402 examples/plot_specgram.py     : t_axis_if_OK(pl.gca(),axt[0:2], np.array(axt[2:])/ffact) 
 1 0.442 examples/get_basic_params.py  : se ValueError('no good shots') ValueError: no good shots 
 2 1.24 examples/tables_extract.py    : OK!
 3 0.306 examples/slider_f_t.py        : ror: [Errno 2] No such file or directory: 'MP512all.npz' 
 4 1.912 examples/check_HD_histogram.py: OK!
 5 1.205 examples/prepfs_range_mp_2.py : OK!
 6 0.3 examples/merge_h1_diagnostics.py:  t_mid = dd['t_mid'] NameError: name 'dd' is not defined 
 7 0.449 examples/mode_identify_script_temp.py:  mode = mode_list[0] IndexError: list index out of range 
 8 0.549 examples/barber_pole.py       : OK!
 9 0.034 examples/example_fragments.py : overlap=220                ^ SyntaxError: invalid syntax 
10 0.448 examples/mode_identify_script.py:  mode = mode_list[0] IndexError: list index out of range 
11 0.153 examples/DA_troubles.py       : OK!
12 0.169 examples/setup_pyfusion.py    : OK!
13 0.261 examples/test_lasso_fs.py     : or key in dd.keys(): NameError: name 'dd' is not defined 
14 1.57 examples/medium_300.py        : OK!
15 0.032 examples/mode_identify_example_from_ICPP_2010.py: ll.txt' skip=4             ^ SyntaxError: invalid syntax 
16 7.488 examples/prepfs_range.py      : OK!
17 4.387 examples/gen_fs_local.py      : OK!
18 2.253 examples/plot_svd.py          : OK!
19 0.788 examples/DP14_mode_plot.py    : OK!
20 0.476 examples/mode_identify_script_old.py: ue') ValueError: Need to choose doN=True and/or doM=True 
21 1.003 examples/plot_signals.py      : OK!
22 2.031 examples/trace_cluster.py     : kh','freq',ind=w23) NameError: name 'w23' is not defined 
23 0.305 examples/copy_DA.py           : ./datamining/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' 
24 0.031 examples/labelled_N_modes.py  : _list=None                 ^ SyntaxError: invalid syntax 
25 7.066 examples/h1_mode_lists.py     : OK!
26 2.171 examples/cluster_info.py      : OK!
27 6.673 examples/gen_fs_bands.py      : :  ########shot 27233: 230/1435 fs finished in 4.96/5.98 
28 2.428 examples/phase_histogram.py   : OK!
29 0.45 examples/find_last_data.py    : .format(l=low))  ValueError: low limit of None not there 
30 1.036 examples/check_HMA_PMA_phase.py: OK!
31 2.252 examples/plot_svd1.py         : OK!
32 0.41 examples/find_close_shot.py   : OK!
33 0.463 examples/realtime_mode.py     : e also cm.gray_r etc NameError: name 'cm' is not defined 
34 0.268 examples/process_chirps.py    :  dfdtunit=dfdtunit)  NameError: name 'dd' is not defined 
35 1.519 examples/cross_phase.py       : OK!
36 0.287 examples/check_DA.py          : /datamining/old/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' 
37 0.083 examples/prepfs_time_mp.py    :  output_path prepfs_time_mp.py: error: too few arguments 
38 2.278 examples/plot_svd_simple.py   : OK!
39 0.264 examples/plot_many.py         : NOT AVAILABLE!!!!!') ValueError: SCAN NOT AVAILABLE!!!!! 
40 0.181 examples/query_h1ds.py        : soup import SqlSoup ImportError: No module named sqlsoup 
41 2.444 examples/plot_basic.py        : vide by zero encountered in true_divide   1/(dy1+dy2),)) 
42 5.397 examples/cluster_phases_three_colour.py: to [10 11 12 13 14 15] - i.e. cldata has different sel,  
43 0.279 examples/new_mode_identify_script.py: ue') ValueError: Need to choose doN=True and/or doM=True 
44 3.465 examples/clean_up_pyfusion_text.py: OK!
45 0.311 examples/test_args.py         : OK!
46 86.342 examples/check_other_data.py  : OK!
47 0.033 examples/find_VSL_locking.py  : ist=mmm                    ^ SyntaxError: invalid syntax 
48 2.412 examples/cluster_DA_0.py      : OK!
49 0.275 examples/plot_text_fft.py     : r: [Errno 2] No such file or directory: 'g:fft26_50.dat' 
50 0.5 examples/merge_text_pyfusion.py: OK!
51 0.946 examples/multi_plot.py        : : igetfile None/wp/wp@105396.dat or .gz, .bz2 not found. 
52 0.036 examples/shauns_clustering.py :         ^ SyntaxError: EOL while scanning string literal 
53 0.288 examples/correct_VSL.py       : ppend_to_DA ImportError: cannot import name append_to_DA 
54 0.266 examples/fft_magnitude_extraction.py:  import HMA_funcs ImportError: No module named HMA_funcs 
55 0.033 examples/good_ones_density_cluster.py: _f32_diags.npz'            ^ SyntaxError: invalid syntax 
56 0.283 examples/plot_shot.py         : OK!
57 0.167 examples/check_dw_pdt.py      : OK!
58 1.255 examples/wid_specgram.py      : h was ['/data/datamining/local_data/', '/g/lhd_shots/']) 
59 7.764 examples/cluster_DA.py        : OK!
60 0.17 examples/check_dw_pdt1.py     : OK!
61 1.223 examples/igetfile.py          : OK!
62 0.281 examples/mode_identify_example_2012.py: orbar=colorbar NameError: name 'colorbar' is not defined 
63 2.397 examples/plot_signals_trivial.py: OK!
64 0.336 examples/small_65.py          : OK!
65 2.765 examples/ne_profile_preprocess.py: OK!
66 0.34 examples/small_300.py         : OK!
67 3.389 examples/fft_magnitude_extraction_LHD.py: OK!
68 0.449 examples/merge_basic_diagnostics.py: rror: [Errno 2] No such file or directory: 'DA_file.npz' 
69 3.497 examples/clean_up_pyfusion_text_mp.py: OK!
70 2.214 examples/plot_svd_old.py      : OK!
71 0.424 examples/merge_basic_HJ_diagnostics.py: sz = len(dd['shot']) NameError: name 'dd' is not defined 
72 0.705 examples/svd_simple_rotator.py: OK!
73 0.447 examples/test_check_buttons1.py: OK!
74 0.464 examples/testall.py           : OK!
75 0.923 examples/save_to_local.py     : OK!
76 4.59 examples/density_cluster.py   : OK!
77 0.389 examples/fonts.py             : OK!
78 0.037 examples/h1_mode_identify.py  : amples/cluster_info.py     ^ SyntaxError: invalid syntax 
79 0.049 examples/prepfs_range_sep.py  : utput_path prepfs_range_sep.py: error: too few arguments 
80 2.627 examples/read_DF.py           : g/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags_comp5.h5...done 
81 0.394 examples/test_check_buttons.py: OK!
82 0.175 examples/__init__.py          : OK!
83 0.168 examples/new_mode_identify_script_0.py: # need to adjust NameError: name 'arange' is not defined 
84 1.662 examples/channel_angle_games.py: OK!
85 2.198 examples/many_specs.py        : r import Node ImportError: No module named node_explorer 
86 2.641 examples/gen_fs_general.py    : OK!
87 0.126 examples/get_kappa.py         : >     kh = 0.*shot NameError: name 'shot' is not defined 
88 0.312 examples/flip_phase.py        : OK!
89 0.285 examples/plot_text_pyfusion.py: /pyfusion/examples/PF2_120227_HMPno12_27233_27233_1.dat' 
90 1.919 examples/check_polarity_flips.py: OK!
91 0.43 examples/play_with_fft_signals.py: OK!
92 0.148 examples/prepfs_range_mp.py   : OK!
42 errors out of 93
When run stand alone, 29 work, 43 errors out of 72

Python3.5 30 dec
 0 1.853 examples/plot_specgram.py     : b'OK!'
 1 0.724 examples/get_basic_params.py  : b"se ValueError('no good shots') ValueError: no good shots "
 2 1.251 examples/tables_extract.py    : b'OK!'
 3 0.431 examples/slider_f_t.py        : b"ror: [Errno 2] No such file or directory: 'MP512all.npz' "
 4 2.001 examples/check_HD_histogram.py: b'OK!'
 5 1.903 examples/prepfs_range_mp_2.py : b'OK!'
 6 0.201 examples/merge_h1_diagnostics.py: b"t__(name, gbls) ImportError: No module named 'mdsdevice' "
 7 0.731 examples/mode_identify_script_temp.py: b'e") LookupError: dd not loaded into memory - can\'t store '
 8 0.607 examples/barber_pole.py       : b'OK!'
 9 0.022 examples/example_fragments.py : b'overlap=220                ^ SyntaxError: invalid syntax '
10 0.753 examples/mode_identify_script.py: b' mode = mode_list[0] IndexError: list index out of range '
11 0.355 examples/DA_troubles.py       : b'OK!'
12 0.274 examples/setup_pyfusion.py    : b"mport.install() ImportError: No module named 'pyximport' "
13 0.471 examples/test_lasso_fs.py     : b"or key in dd.keys(): NameError: name 'dd' is not defined "
14 1.808 examples/medium_300.py        : b'OK!'
15 0.024 examples/mode_identify_example_from_ICPP_2010.py: b"ll.txt' skip=4             ^ SyntaxError: invalid syntax "
16 12.941 examples/prepfs_range.py      : b'OK!'
17 7.464 examples/gen_fs_local.py      : b'OK!'
18 0.812 examples/plot_svd.py          : b"t__(name, gbls) ImportError: No module named 'mdsdevice' "
19 0.746 examples/DP14_mode_plot.py    : b'OK!'
20 0.747 examples/mode_identify_script_old.py: b'e") LookupError: dd not loaded into memory - can\'t store '
21 0.912 examples/plot_signals.py      : b'OK!'
22 0.703 examples/trace_cluster.py     : b"ile(filename,'r')) NameError: name 'file' is not defined "
23 0.588 examples/copy_DA.py           : b"./datamining/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' "
24 0.033 examples/labelled_N_modes.py  : b'_list=None                 ^ SyntaxError: invalid syntax '
25 0.689 examples/h1_mode_lists.py     : b"ile(filename,'r')) NameError: name 'file' is not defined "
26 0.679 examples/cluster_info.py      : b"ile(filename,'r')) NameError: name 'file' is not defined "
27 11.343 examples/gen_fs_bands.py      : b'  ########shot 27233: 230/1435 fs finished in 9.77/10.48 '
28 1.982 examples/phase_histogram.py   : b'OK!'
29 0.748 examples/find_last_data.py    : b'.format(l=low))  ValueError: low limit of None not there '
30 0.816 examples/check_HMA_PMA_phase.py: b"t__(name, gbls) ImportError: No module named 'mdsdevice' "
31 0.023 examples/plot_svd1.py         : b'                           ^ SyntaxError: invalid syntax '
32 0.665 examples/find_close_shot.py   : b' You may need to pass the encoding= option to numpy.load '
33 0.537 examples/realtime_mode.py     : b"t__(name, gbls) ImportError: No module named 'mdsdevice' "
34 0.474 examples/process_chirps.py    : b" dfdtunit=dfdtunit)  NameError: name 'dd' is not defined "
35 0.799 examples/cross_phase.py       : b"t__(name, gbls) ImportError: No module named 'mdsdevice' "
36 0.484 examples/check_DA.py          : b"/datamining/old/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' "
37 0.151 examples/prepfs_time_mp.py    : b'error: the following arguments are required: output_path '
38 0.821 examples/plot_svd_simple.py   : b"t__(name, gbls) ImportError: No module named 'mdsdevice' "
39 0.02 examples/plot_many.py         : b'(),shot_list))             ^ SyntaxError: invalid syntax '
40 0.416 examples/query_h1ds.py        : b"up ImportError: No module named 'sqlalchemy.ext.sqlsoup' "
41 0.851 examples/plot_basic.py        : b'alid mode: %r" % (mode,)) ValueError: Invalid mode: \'rU\' '
42 0.486 examples/cluster_phases_three_colour.py: b"wer() != 'n': NameError: name 'raw_input' is not defined "
43 0.353 examples/new_mode_identify_script.py: b"esis.pickle','r')) NameError: name 'file' is not defined "
44 9.433 examples/clean_up_pyfusion_text.py: b'OK!'
45 0.511 examples/test_args.py         : b'OK!'
46 0.021 examples/check_other_data.py  : b'rror: inconsistent use of tabs and spaces in indentation '
47 0.022 examples/find_VSL_locking.py  : b'ist=mmm                    ^ SyntaxError: invalid syntax '
48 3.154 examples/cluster_DA_0.py      : b'OK!'
49 0.5 examples/plot_text_fft.py     : b"r: [Errno 2] No such file or directory: 'g:fft26_50.dat' "
50 0.987 examples/merge_text_pyfusion.py: b'OK!'
51 1.031 examples/multi_plot.py        : b': igetfile None/wp/wp@105396.dat or .gz, .bz2 not found. '
52 0.024 examples/shauns_clustering.py : b'        ^ SyntaxError: EOL while scanning string literal '
53 0.483 examples/correct_VSL.py       : b"end_to_DA ImportError: cannot import name 'append_to_DA' "
54 0.022 examples/fft_magnitude_extraction.py: b"   ^ SyntaxError: Missing parentheses in call to 'print' "
55 0.022 examples/good_ones_density_cluster.py: b"_f32_diags.npz'            ^ SyntaxError: invalid syntax "
56 0.477 examples/plot_shot.py         : b'OK!'
57 0.354 examples/check_dw_pdt.py      : b'OK!'
58 0.021 examples/wid_specgram.py      : b'p, _type, fmod             ^ SyntaxError: invalid syntax '
59 7.451 examples/cluster_DA.py        : b'rnings.warn("Setting the \'color\' property will override" '
60 0.359 examples/check_dw_pdt1.py     : b'OK!'
61 1.182 examples/igetfile.py          : b'OK!'
62 0.497 examples/mode_identify_example_2012.py: b"orbar=colorbar NameError: name 'colorbar' is not defined "
63 1.61 examples/plot_signals_trivial.py: b'OK!'
64 0.553 examples/small_65.py          : b'OK!'
65 0.021 examples/ne_profile_preprocess.py: b'                           ^ SyntaxError: invalid syntax '
66 0.542 examples/small_300.py         : b'OK!'
67 0.022 examples/fft_magnitude_extraction_LHD.py: b"   ^ SyntaxError: Missing parentheses in call to 'print' "
68 0.754 examples/merge_basic_diagnostics.py: b"rror: [Errno 2] No such file or directory: 'DA_file.npz' "
69 9.431 examples/clean_up_pyfusion_text_mp.py: b'OK!'
70 0.024 examples/plot_svd_old.py      : b'                           ^ SyntaxError: invalid syntax '
71 0.74 examples/merge_basic_HJ_diagnostics.py: b"sz = len(dd['shot']) NameError: name 'dd' is not defined "
72 0.021 examples/svd_simple_rotator.py: b'hronof)))                  ^ SyntaxError: invalid syntax '
73 0.505 examples/test_check_buttons1.py: b'OK!'
74 0.841 examples/testall.py           : b"wer() != 'n': NameError: name 'raw_input' is not defined "
75 0.898 examples/save_to_local.py     : b") TypeError: type Signal doesn't define __round__ method "
76 0.492 examples/density_cluster.py   : b"AttributeError: 'range' object has no attribute 'extend' "
77 0.44 examples/fonts.py             : b'OK!'
78 0.035 examples/h1_mode_identify.py  : b'amples/cluster_info.py     ^ SyntaxError: invalid syntax '
79 0.045 examples/prepfs_range_sep.py  : b'error: the following arguments are required: output_path '
80 3.107 examples/read_DF.py           : b'g/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags_comp5.h5...done '
81 0.483 examples/test_check_buttons.py: b'OK!'
82 0.37 examples/__init__.py          : b'OK!'
83 0.352 examples/new_mode_identify_script_0.py: b"# need to adjust NameError: name 'arange' is not defined "
84 1.282 examples/channel_angle_games.py: b'OK!'
85 0.026 examples/many_specs.py        : b"   ^ SyntaxError: Missing parentheses in call to 'print' "
86 0.834 examples/gen_fs_general.py    : b'd \'mdsdevice\'",)   (shot,sys.exc_info()[1].__repr__()))) '
87 0.194 examples/get_kappa.py         : b"t__(name, gbls) ImportError: No module named 'mdsdevice' "
88 0.508 examples/flip_phase.py        : b'OK!'
89 0.022 examples/plot_text_pyfusion.py: b'                           ^ SyntaxError: invalid syntax '
90 1.636 examples/check_polarity_flips.py: b'OK!'
91 0.761 examples/play_with_fft_signals.py: b'OK!'
92 0.172 examples/prepfs_range_mp.py   : b"tributeError: module 'itertools' has no attribute 'izip' "

61 errors out of 93


This file was run on Wed 2 dec, - future ones will be prepended.
==python 2.7.6==
examples/plot_specgram.py     : t_axis_if_OK(pl.gca(),axt[0:2], np.array(axt[2:])/ffact) 
examples/get_basic_params.py  : se ValueError('no good shots') ValueError: no good shots 
examples/slider_f_t.py        : ror: [Errno 2] No such file or directory: 'MP512all.npz' 
examples/check_HD_histogram.py: OK!
examples/prepfs_range_mp_2.py : OK!
examples/merge_h1_diagnostics.py:  t_mid = dd['t_mid'] NameError: name 'dd' is not defined 
examples/mode_identify_script_temp.py:  mode = mode_list[0] IndexError: list index out of range 
examples/barber_pole.py       : OK!
examples/example_fragments.py : overlap=220                ^ SyntaxError: invalid syntax 
examples/mode_identify_script.py:  mode = mode_list[0] IndexError: list index out of range 
examples/DA_troubles.py       : OK!
examples/setup_pyfusion.py    : OK!
examples/test_lasso_fs.py     : or key in dd.keys(): NameError: name 'dd' is not defined 
examples/medium_300.py        : OK!
examples/mode_identify_example_from_ICPP_2010.py: ll.txt' skip=4             ^ SyntaxError: invalid syntax 
examples/prepfs_range.py      : OK!
examples/gen_fs_local.py      : OK!
examples/plot_svd.py          : OK!
examples/DP14_mode_plot.py    : OK!
examples/mode_identify_script_old.py: ue') ValueError: Need to choose doN=True and/or doM=True 
examples/plot_signals.py      : OK!
examples/trace_cluster.py     : kh','freq',ind=w23) NameError: name 'w23' is not defined 
examples/copy_DA.py           : ./datamining/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' 
examples/labelled_N_modes.py  : _list=None                 ^ SyntaxError: invalid syntax 
examples/h1_mode_lists.py     : OK!
examples/cluster_info.py      : OK!
examples/gen_fs_bands.py      : :  ########shot 27233: 230/1435 fs finished in 5.79/7.53 
examples/phase_histogram.py   : = len(phases[0]) NameError: name 'phases' is not defined 
examples/find_last_data.py    : .format(l=low))  ValueError: low limit of None not there 
examples/check_HMA_PMA_phase.py: OK!
examples/plot_svd1.py         : OK!
examples/find_close_shot.py   : ebug=0, verbose=1): NameError: name 'LHD' is not defined 
examples/realtime_mode.py     : e also cm.gray_r etc NameError: name 'cm' is not defined 
examples/process_chirps.py    :  dfdtunit=dfdtunit)  NameError: name 'dd' is not defined 
examples/cross_phase.py       : OK!
examples/check_DA.py          : /datamining/old/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' 
examples/prepfs_time_mp.py    :  output_path prepfs_time_mp.py: error: too few arguments 
examples/plot_svd_simple.py   : OK!
examples/plot_many.py         : NOT AVAILABLE!!!!!') ValueError: SCAN NOT AVAILABLE!!!!! 
examples/query_h1ds.py        : soup import SqlSoup ImportError: No module named sqlsoup 
examples/plot_basic.py        : vide by zero encountered in true_divide   1/(dy1+dy2),)) 
examples/cluster_phases_three_colour.py: to [10 11 12 13 14 15] - i.e. cldata has different sel,  
examples/new_mode_identify_script.py: ue') ValueError: Need to choose doN=True and/or doM=True 
examples/clean_up_pyfusion_text.py: OK!
examples/check_other_data.py  : OK!
examples/find_VSL_locking.py  : ist=mmm                    ^ SyntaxError: invalid syntax 
examples/plot_text_fft.py     : r: [Errno 2] No such file or directory: 'g:fft26_50.dat' 
examples/merge_text_pyfusion.py: OK!
examples/multi_plot.py        : atamining/cache/ip/ip@105396.dat or .gz, .bz2 not found. 
examples/shauns_clustering.py :         ^ SyntaxError: EOL while scanning string literal 
examples/correct_VSL.py       : ppend_to_DA ImportError: cannot import name append_to_DA 
examples/fft_magnitude_extraction.py:  import HMA_funcs ImportError: No module named HMA_funcs 
examples/good_ones_density_cluster.py: _f32_diags.npz'            ^ SyntaxError: invalid syntax 
examples/plot_shot.py         : OK!
examples/check_dw_pdt.py      : OK!
examples/wid_specgram.py      : h was ['/data/datamining/local_data/', '/g/lhd_shots/']) 
examples/cluster_DA.py        : OK!
examples/check_dw_pdt1.py     : OK!
examples/igetfile.py          : OK!
examples/mode_identify_example_2012.py: orbar=colorbar NameError: name 'colorbar' is not defined 
examples/plot_signals_trivial.py: OK!
examples/small_65.py          : OK!
examples/ne_profile_preprocess.py: OK!
examples/small_300.py         : OK!
examples/fft_magnitude_extraction_LHD.py: OK!
examples/merge_basic_diagnostics.py: rror: [Errno 2] No such file or directory: 'DA_file.npz' 
examples/clean_up_pyfusion_text_mp.py: OK!
examples/plot_svd_old.py      : OK!
examples/merge_basic_HJ_diagnostics.py: data ImportError: numpy.core.multiarray failed to import 
examples/svd_simple_rotator.py: OK!
examples/test_check_buttons1.py: OK!
examples/testall.py           : OK!
examples/save_to_local.py     : OK!
examples/density_cluster.py   : s,method=method) NameError: name 'phases' is not defined 
examples/fonts.py             : OK!
examples/h1_mode_identify.py  : amples/cluster_info.py     ^ SyntaxError: invalid syntax 
examples/prepfs_range_sep.py  : utput_path prepfs_range_sep.py: error: too few arguments 
examples/read_DF.py           : ess_cmd_line_args ImportError: No module named bdb_utils 
examples/test_check_buttons.py: OK!
examples/__init__.py          : OK!
examples/new_mode_identify_script_0.py: # need to adjust NameError: name 'arange' is not defined 
examples/channel_angle_games.py: OK!
examples/many_specs.py        : r import Node ImportError: No module named node_explorer 
examples/gen_fs_general.py    : OK!
examples/get_kappa.py         : >     kh = 0.*shot NameError: name 'shot' is not defined 
examples/flip_phase.py        : b_0']))[0];len(wnan) NameError: name 'dd' is not defined 
examples/plot_text_pyfusion.py: /pyfusion/examples/PF2_120227_HMPno12_27233_27233_1.dat' 
examples/check_polarity_flips.py: OK!
examples/play_with_fft_signals.py: OK!
examples/prepfs_range_mp.py   : OK!
46 errors out of 90

========= python 2.7.6 -3 ===========
examples/plot_specgram.py     : psule objects instead.   self._renderer.draw_text_image( 
examples/get_basic_params.py  : se ValueError('no good shots') ValueError: no good shots 
examples/slider_f_t.py        : ror: [Errno 2] No such file or directory: 'MP512all.npz' 
examples/check_HD_histogram.py:   self._renderer.draw_path(gc, path, transform, rgbFace) 
examples/prepfs_range_mp_2.py : umpy 1.9   warnings.warn(_msg, ModuleDeprecationWarning) 
examples/merge_h1_diagnostics.py:  t_mid = dd['t_mid'] NameError: name 'dd' is not defined 
examples/mode_identify_script_temp.py:  mode = mode_list[0] IndexError: list index out of range 
examples/barber_pole.py       : psule objects instead.   self._renderer.draw_text_image( 
examples/example_fragments.py : overlap=220                ^ SyntaxError: invalid syntax 
examples/mode_identify_script.py:  mode = mode_list[0] IndexError: list index out of range 
examples/DA_troubles.py       : backends._backend_agg import RendererAgg as _RendererAgg 
examples/setup_pyfusion.py    : umpy 1.9   warnings.warn(_msg, ModuleDeprecationWarning) 
examples/test_lasso_fs.py     : or key in dd.keys(): NameError: name 'dd' is not defined 
examples/medium_300.py        : backends._backend_agg import RendererAgg as _RendererAgg 
examples/mode_identify_example_from_ICPP_2010.py: ll.txt' skip=4             ^ SyntaxError: invalid syntax 
examples/prepfs_range.py      : OK!
examples/gen_fs_local.py      : super(MetaMethods, cls).__new__(cls, name, bases, attrs) 
examples/plot_svd.py          : enderer.draw_text_image(font_image, x, y + 1, angle, gc) 
examples/DP14_mode_plot.py    : psule objects instead.   self._renderer.draw_text_image( 
examples/mode_identify_script_old.py: ue') ValueError: Need to choose doN=True and/or doM=True 
examples/plot_signals.py      : psule objects instead.   self._renderer.draw_text_image( 
examples/trace_cluster.py     : kh','freq',ind=w23) NameError: name 'w23' is not defined 
examples/copy_DA.py           : ./datamining/DA/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' 
examples/labelled_N_modes.py  : _list=None                 ^ SyntaxError: invalid syntax 
examples/h1_mode_lists.py     : psule objects instead.   self._renderer.draw_text_image( 
examples/cluster_info.py      : se use capsule objects instead.   from . import dfitpack 
examples/gen_fs_bands.py      : :  ########shot 27233: 230/1435 fs finished in 5.88/7.70 
examples/phase_histogram.py   : = len(phases[0]) NameError: name 'phases' is not defined 
examples/find_last_data.py    : .format(l=low))  ValueError: low limit of None not there 
examples/check_HMA_PMA_phase.py: psule objects instead.   self._renderer.draw_text_image( 
examples/plot_svd1.py         : enderer.draw_text_image(font_image, x, y + 1, angle, gc) 
examples/find_close_shot.py   : ebug=0, verbose=1): NameError: name 'LHD' is not defined 
examples/realtime_mode.py     : e also cm.gray_r etc NameError: name 'cm' is not defined 
examples/process_chirps.py    :  dfdtunit=dfdtunit)  NameError: name 'dd' is not defined 
examples/cross_phase.py       : psule objects instead.   self._renderer.draw_text_image( 
examples/check_DA.py          : /datamining/old/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' 
examples/prepfs_time_mp.py    :  output_path prepfs_time_mp.py: error: too few arguments 
examples/plot_svd_simple.py   : enderer.draw_text_image(font_image, x, y + 1, angle, gc) 
examples/plot_many.py         : NOT AVAILABLE!!!!!') ValueError: SCAN NOT AVAILABLE!!!!! 
examples/query_h1ds.py        : soup import SqlSoup ImportError: No module named sqlsoup 
examples/plot_basic.py        : psule objects instead.   self._renderer.draw_text_image( 
examples/cluster_phases_three_colour.py: enderer.draw_text_image(font_image, x, y + 1, angle, gc) 
examples/new_mode_identify_script.py: ue') ValueError: Need to choose doN=True and/or doM=True 
examples/clean_up_pyfusion_text.py: backends._backend_agg import RendererAgg as _RendererAgg 
examples/check_other_data.py  : psule objects instead.   self._renderer.draw_text_image( 
examples/find_VSL_locking.py  : ist=mmm                    ^ SyntaxError: invalid syntax 
examples/plot_text_fft.py     : r: [Errno 2] No such file or directory: 'g:fft26_50.dat' 
examples/merge_text_pyfusion.py: backends._backend_agg import RendererAgg as _RendererAgg 
examples/multi_plot.py        : atamining/cache/ip/ip@105396.dat or .gz, .bz2 not found. 
examples/shauns_clustering.py :         ^ SyntaxError: EOL while scanning string literal 
examples/correct_VSL.py       : ppend_to_DA ImportError: cannot import name append_to_DA 
examples/fft_magnitude_extraction.py:  import HMA_funcs ImportError: No module named HMA_funcs 
examples/good_ones_density_cluster.py: _f32_diags.npz'            ^ SyntaxError: invalid syntax 
examples/plot_shot.py         : umpy 1.9   warnings.warn(_msg, ModuleDeprecationWarning) 
examples/check_dw_pdt.py      : backends._backend_agg import RendererAgg as _RendererAgg 
examples/wid_specgram.py      : h was ['/data/datamining/local_data/', '/g/lhd_shots/']) 
examples/cluster_DA.py        : upported in 3.x; use except clauses   self.__exc_clear() 
examples/check_dw_pdt1.py     : backends._backend_agg import RendererAgg as _RendererAgg 
examples/igetfile.py          : psule objects instead.   self._renderer.draw_text_image( 
examples/mode_identify_example_2012.py: orbar=colorbar NameError: name 'colorbar' is not defined 
examples/plot_signals_trivial.py: psule objects instead.   self._renderer.draw_text_image( 
examples/small_65.py          : backends._backend_agg import RendererAgg as _RendererAgg 
examples/ne_profile_preprocess.py: .toCtype() * int(descr.arsize/descr.length))).contents)) 
examples/small_300.py         : backends._backend_agg import RendererAgg as _RendererAgg 
examples/fft_magnitude_extraction_LHD.py: instead.   return self._renderer.draw_markers(*kl, **kw) 
examples/merge_basic_diagnostics.py: rror: [Errno 2] No such file or directory: 'DA_file.npz' 
examples/clean_up_pyfusion_text_mp.py: backends._backend_agg import RendererAgg as _RendererAgg 
examples/plot_svd_old.py      : enderer.draw_text_image(font_image, x, y + 1, angle, gc) 
examples/merge_basic_HJ_diagnostics.py: data ImportError: numpy.core.multiarray failed to import 
examples/svd_simple_rotator.py: psule objects instead.   self._renderer.draw_text_image( 
examples/test_check_buttons1.py: psule objects instead.   self._renderer.draw_text_image( 
examples/testall.py           : se use capsule objects instead.   from . import dfitpack 
examples/save_to_local.py     : super(MetaMethods, cls).__new__(cls, name, bases, attrs) 
examples/density_cluster.py   : s,method=method) NameError: name 'phases' is not defined 
examples/fonts.py             : psule objects instead.   self._renderer.draw_text_image( 
examples/h1_mode_identify.py  : amples/cluster_info.py     ^ SyntaxError: invalid syntax 
examples/prepfs_range_sep.py  : utput_path prepfs_range_sep.py: error: too few arguments 
examples/read_DF.py           : ess_cmd_line_args ImportError: No module named bdb_utils 
examples/test_check_buttons.py: psule objects instead.   self._renderer.draw_text_image( 
examples/__init__.py          : backends._backend_agg import RendererAgg as _RendererAgg 
examples/new_mode_identify_script_0.py: # need to adjust NameError: name 'arange' is not defined 
examples/channel_angle_games.py: psule objects instead.   self._renderer.draw_text_image( 
examples/many_specs.py        : r import Node ImportError: No module named node_explorer 
examples/gen_fs_general.py    : .toCtype() * int(descr.arsize/descr.length))).contents)) 
examples/get_kappa.py         : >     kh = 0.*shot NameError: name 'shot' is not defined 
examples/flip_phase.py        : b_0']))[0];len(wnan) NameError: name 'dd' is not defined 
examples/plot_text_pyfusion.py: /pyfusion/examples/PF2_120227_HMPno12_27233_27233_1.dat' 
examples/check_polarity_flips.py: psule objects instead.   self._renderer.draw_text_image( 
examples/play_with_fft_signals.py: super(MetaMethods, cls).__new__(cls, name, bases, attrs) 
examples/prepfs_range_mp.py   : OK!
88 errors out of 90
===================
python 3 warnings coming from my files
0 pyfusion/examples/plot_specgram.py
16 pyfusion/examples/gen_fs_local.py
17 pyfusion/examples/plot_svd.py
20 pyfusion/examples/plot_signals.py
22 pyfusion/examples/copy_DA.py
26 pyfusion/examples/gen_fs_bands.py
28 pyfusion/examples/find_last_data.py
29 pyfusion/examples/check_HMA_PMA_phase.py
30 pyfusion/examples/plot_svd1.py
32 pyfusion/examples/realtime_mode.py
34 pyfusion/examples/cross_phase.py
35 pyfusion/examples/check_DA.py
37 pyfusion/examples/plot_svd_simple.py
38 pyfusion/examples/plot_many.py
40 pyfusion/examples/plot_basic.py
41 pyfusion/examples/cluster_phases_three_colour.py
48 pyfusion/examples/multi_plot.py
55 pyfusion/examples/wid_specgram.py
56 pyfusion/examples/cluster_DA.py
60 pyfusion/examples/plot_signals_trivial.py
62 pyfusion/examples/ne_profile_preprocess.py
64 pyfusion/examples/fft_magnitude_extraction_LHD.py
65 pyfusion/examples/merge_basic_diagnostics.py
67 pyfusion/examples/plot_svd_old.py
68 pyfusion/examples/merge_basic_HJ_diagnostics.py
72 pyfusion/examples/save_to_local.py
81 pyfusion/examples/channel_angle_games.py
83 pyfusion/examples/gen_fs_general.py
86 pyfusion/examples/plot_text_pyfusion.py
87 pyfusion/examples/check_polarity_flips.py
88 pyfusion/examples/play_with_fft_signals.py

(3, 'pyfusion/examples/check_HD_histogram.py')
(17, 'pyfusion/examples/plot_svd.py')
(26, 'pyfusion/examples/gen_fs_bands.py')
(29, 'pyfusion/examples/check_HMA_PMA_phase.py')
(30, 'pyfusion/examples/plot_svd1.py')
(32, 'pyfusion/examples/realtime_mode.py')
(34, 'pyfusion/examples/cross_phase.py')
(37, 'pyfusion/examples/plot_svd_simple.py')
(41, 'pyfusion/examples/cluster_phases_three_colour.py')
(62, 'pyfusion/examples/ne_profile_preprocess.py')
(64, 'pyfusion/examples/fft_magnitude_extraction_LHD.py')
(67, 'pyfusion/examples/plot_svd_old.py')
(69, 'pyfusion/examples/svd_simple_rotator.py')
(82, 'pyfusion/examples/many_specs.py')
(83, 'pyfusion/examples/gen_fs_general.py')

=============== V2.7 Anaconda =========
examples/plot_specgram.py     : t_axis_if_OK(pl.gca(),axt[0:2], np.array(axt[2:])/ffact) 
examples/get_basic_params.py  : se ValueError('no good shots') ValueError: no good shots 
examples/slider_f_t.py        : ror: [Errno 2] No such file or directory: 'MP512all.npz' 
examples/check_HD_histogram.py: ngs will not be saved or shared with other applications. 
examples/prepfs_range_mp_2.py : OK!
examples/merge_h1_diagnostics.py: iconda/bin/../lib/libreadline.so.6: undefined symbol: PC 
examples/mode_identify_script_temp.py:  mode = mode_list[0] IndexError: list index out of range 
examples/barber_pole.py       : ngs will not be saved or shared with other applications. 
examples/example_fragments.py : overlap=220                ^ SyntaxError: invalid syntax 
examples/mode_identify_script.py:  mode = mode_list[0] IndexError: list index out of range 
examples/DA_troubles.py       : OK!
examples/setup_pyfusion.py    : OK!
examples/test_lasso_fs.py     : or key in dd.keys(): NameError: name 'dd' is not defined 
examples/medium_300.py        : OK!
examples/mode_identify_example_from_ICPP_2010.py: ll.txt' skip=4             ^ SyntaxError: invalid syntax 
examples/prepfs_range.py      : OK!
examples/gen_fs_local.py      : OK!
examples/plot_svd.py          : iconda/bin/../lib/libreadline.so.6: undefined symbol: PC 
examples/DP14_mode_plot.py    : entwise comparison   if self._edgecolors == str('face'): 
examples/mode_identify_script_old.py: ue') ValueError: Need to choose doN=True and/or doM=True 
examples/plot_signals.py      : ngs will not be saved or shared with other applications. 
examples/trace_cluster.py     : kh','freq',ind=w23) NameError: name 'w23' is not defined 
examples/copy_DA.py           :  import tables as tb ImportError: No module named tables 
examples/labelled_N_modes.py  : _list=None                 ^ SyntaxError: invalid syntax 
examples/h1_mode_lists.py     : ngs will not be saved or shared with other applications. 
examples/cluster_info.py      : OK!
examples/gen_fs_bands.py      :         (actually not- bdb)^ SyntaxError: invalid syntax 
examples/phase_histogram.py   : = len(phases[0]) NameError: name 'phases' is not defined 
examples/find_last_data.py    : .format(l=low))  ValueError: low limit of None not there 
examples/check_HMA_PMA_phase.py: iconda/bin/../lib/libreadline.so.6: undefined symbol: PC 
examples/plot_svd1.py         : iconda/bin/../lib/libreadline.so.6: undefined symbol: PC 
examples/find_close_shot.py   : ebug=0, verbose=1): NameError: name 'LHD' is not defined 
examples/realtime_mode.py     : iconda/bin/../lib/libreadline.so.6: undefined symbol: PC 
examples/process_chirps.py    :  dfdtunit=dfdtunit)  NameError: name 'dd' is not defined 
examples/cross_phase.py       : iconda/bin/../lib/libreadline.so.6: undefined symbol: PC 
examples/check_DA.py          : /datamining/old/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' 
examples/prepfs_time_mp.py    :  output_path prepfs_time_mp.py: error: too few arguments 
examples/plot_svd_simple.py   : iconda/bin/../lib/libreadline.so.6: undefined symbol: PC 
examples/plot_many.py         : NOT AVAILABLE!!!!!') ValueError: SCAN NOT AVAILABLE!!!!! 
examples/query_h1ds.py        : soup import SqlSoup ImportError: No module named sqlsoup 
examples/plot_basic.py        : ngs will not be saved or shared with other applications. 
examples/cluster_phases_three_colour.py: ngs will not be saved or shared with other applications. 
examples/new_mode_identify_script.py: ue') ValueError: Need to choose doN=True and/or doM=True 
examples/clean_up_pyfusion_text.py: OK!
examples/check_other_data.py  : ngs will not be saved or shared with other applications. 
examples/find_VSL_locking.py  : ist=mmm                    ^ SyntaxError: invalid syntax 
examples/plot_text_fft.py     : r: [Errno 2] No such file or directory: 'g:fft26_50.dat' 
examples/merge_text_pyfusion.py: OK!
examples/multi_plot.py        : atamining/cache/ip/ip@105396.dat or .gz, .bz2 not found. 
examples/shauns_clustering.py :         ^ SyntaxError: EOL while scanning string literal 
examples/correct_VSL.py       : ppend_to_DA ImportError: cannot import name append_to_DA 
examples/fft_magnitude_extraction.py:  import HMA_funcs ImportError: No module named HMA_funcs 
examples/good_ones_density_cluster.py: _f32_diags.npz'            ^ SyntaxError: invalid syntax 
examples/plot_shot.py         : OK!
examples/check_dw_pdt.py      : OK!
examples/wid_specgram.py      :  require Tix') _tkinter.TclError: can't find package Tix 
examples/cluster_DA.py        : OK!
examples/check_dw_pdt1.py     : OK!
examples/igetfile.py          : ngs will not be saved or shared with other applications. 
examples/mode_identify_example_2012.py: orbar=colorbar NameError: name 'colorbar' is not defined 
examples/plot_signals_trivial.py: ngs will not be saved or shared with other applications. 
examples/small_65.py          : OK!
examples/ne_profile_preprocess.py: iconda/bin/../lib/libreadline.so.6: undefined symbol: PC 
examples/small_300.py         : OK!
examples/fft_magnitude_extraction_LHD.py: entwise comparison   if self._edgecolors == str('face'): 
examples/merge_basic_diagnostics.py: rror: [Errno 2] No such file or directory: 'DA_file.npz' 
examples/clean_up_pyfusion_text_mp.py: OK!
examples/plot_svd_old.py      : iconda/bin/../lib/libreadline.so.6: undefined symbol: PC 
examples/merge_basic_HJ_diagnostics.py: sz = len(dd['shot']) NameError: name 'dd' is not defined 
examples/svd_simple_rotator.py: ngs will not be saved or shared with other applications. 
examples/test_check_buttons1.py: ngs will not be saved or shared with other applications. 
examples/testall.py           : OK!
examples/save_to_local.py     : OK!
examples/density_cluster.py   : s,method=method) NameError: name 'phases' is not defined 
examples/fonts.py             : ngs will not be saved or shared with other applications. 
examples/h1_mode_identify.py  : amples/cluster_info.py     ^ SyntaxError: invalid syntax 
examples/prepfs_range_sep.py  : utput_path prepfs_range_sep.py: error: too few arguments 
examples/read_DF.py           :  import tables as tb ImportError: No module named tables 
examples/test_check_buttons.py: ngs will not be saved or shared with other applications. 
examples/__init__.py          : OK!
examples/new_mode_identify_script_0.py: # need to adjust NameError: name 'arange' is not defined 
examples/channel_angle_games.py: ngs will not be saved or shared with other applications. 
examples/many_specs.py        : iconda/bin/../lib/libreadline.so.6: undefined symbol: PC 
examples/gen_fs_general.py    : ed symbol: PC',)   (shot,sys.exc_info()[1].__repr__()))) 
examples/get_kappa.py         : iconda/bin/../lib/libreadline.so.6: undefined symbol: PC 
examples/flip_phase.py        : b_0']))[0];len(wnan) NameError: name 'dd' is not defined 
examples/plot_text_pyfusion.py: /pyfusion/examples/PF2_120227_HMPno12_27233_27233_1.dat' 
examples/check_polarity_flips.py: ngs will not be saved or shared with other applications. 
examples/play_with_fft_signals.py: OK!
examples/prepfs_range_mp.py   : OK!

69 errors out of 90


=========== Python 3.4 anaconda ========== (3.5 was 65, but I improved since then)
examples/plot_specgram.py     : b'OK!'
examples/get_basic_params.py  : b" import StringIO ImportError: No module named 'StringIO' "
examples/slider_f_t.py        : b"ror: [Errno 2] No such file or directory: 'MP512all.npz' "
examples/check_HD_histogram.py: b'OK!'
examples/prepfs_range_mp_2.py : b'OK!'
examples/merge_h1_diagnostics.py: b"port__(name, gbls) ImportError: No module named 'mdsdcl' "
examples/mode_identify_script_temp.py: b'e") LookupError: dd not loaded into memory - can\'t store '
examples/barber_pole.py       : b'OK!'
examples/example_fragments.py : b'overlap=220                ^ SyntaxError: invalid syntax '
examples/mode_identify_script.py: b' mode = mode_list[0] IndexError: list index out of range '
examples/DA_troubles.py       : b'OK!'
examples/setup_pyfusion.py    : b"mport.install() ImportError: No module named 'pyximport' "
examples/test_lasso_fs.py     : b"or key in dd.keys(): NameError: name 'dd' is not defined "
examples/medium_300.py        : b'OK!'
examples/mode_identify_example_from_ICPP_2010.py: b"ll.txt' skip=4             ^ SyntaxError: invalid syntax "
examples/prepfs_range.py      : b'OK!'
examples/gen_fs_local.py      : b'OK!'
examples/plot_svd.py          : b"port__(name, gbls) ImportError: No module named 'mdsdcl' "
examples/DP14_mode_plot.py    : b'OK!'
examples/mode_identify_script_old.py: b'e") LookupError: dd not loaded into memory - can\'t store '
examples/plot_signals.py      : b'OK!'
examples/trace_cluster.py     : b"ile(filename,'r')) NameError: name 'file' is not defined "
examples/copy_DA.py           : b"mport tables as tb ImportError: No module named 'tables' "
examples/labelled_N_modes.py  : b'_list=None                 ^ SyntaxError: invalid syntax '
examples/h1_mode_lists.py     : b"ile(filename,'r')) NameError: name 'file' is not defined "
examples/cluster_info.py      : b"ile(filename,'r')) NameError: name 'file' is not defined "
examples/gen_fs_bands.py      : b' Time:  ########shot 27233: 4/8 fs finished in 0.04/0.87 '
examples/phase_histogram.py   : b"= len(phases[0]) NameError: name 'phases' is not defined "
examples/find_last_data.py    : b'.format(l=low))  ValueError: low limit of None not there '
examples/check_HMA_PMA_phase.py: b"port__(name, gbls) ImportError: No module named 'mdsdcl' "
examples/plot_svd1.py         : b'                           ^ SyntaxError: invalid syntax '
examples/find_close_shot.py   : b"ebug=0, verbose=1): NameError: name 'LHD' is not defined "
examples/realtime_mode.py     : b"port__(name, gbls) ImportError: No module named 'mdsdcl' "
examples/process_chirps.py    : b" dfdtunit=dfdtunit)  NameError: name 'dd' is not defined "
examples/cross_phase.py       : b"port__(name, gbls) ImportError: No module named 'mdsdcl' "
examples/check_DA.py          : b"/datamining/old/PF2_130813_50_5X_1.5_5b_rms_1_diags.npz' "
examples/prepfs_time_mp.py    : b'error: the following arguments are required: output_path '
examples/plot_svd_simple.py   : b"port__(name, gbls) ImportError: No module named 'mdsdcl' "
examples/plot_many.py         : b'(),shot_list))             ^ SyntaxError: invalid syntax '
examples/query_h1ds.py        : b" create_engine ImportError: No module named 'sqlalchemy' "
examples/plot_basic.py        : b"AttributeError: 'dict' object has no attribute 'has_key' "
examples/cluster_phases_three_colour.py: b"wer() != 'n': NameError: name 'raw_input' is not defined "
examples/new_mode_identify_script.py: b"esis.pickle','r')) NameError: name 'file' is not defined "
examples/clean_up_pyfusion_text.py: b'OK!'
examples/check_other_data.py  : b'rror: inconsistent use of tabs and spaces in indentation '
examples/find_VSL_locking.py  : b'ist=mmm                    ^ SyntaxError: invalid syntax '
examples/plot_text_fft.py     : b"r: [Errno 2] No such file or directory: 'g:fft26_50.dat' "
examples/merge_text_pyfusion.py: b"yfusion/acquisition/PF2_121206_54185_384_rms_1.dat.bz2'] "
examples/multi_plot.py        : b" import StringIO ImportError: No module named 'StringIO' "
examples/shauns_clustering.py : b'        ^ SyntaxError: EOL while scanning string literal '
examples/correct_VSL.py       : b"end_to_DA ImportError: cannot import name 'append_to_DA' "
examples/fft_magnitude_extraction.py: b"   ^ SyntaxError: Missing parentheses in call to 'print' "
examples/good_ones_density_cluster.py: b"_f32_diags.npz'            ^ SyntaxError: invalid syntax "
examples/plot_shot.py         : b'OK!'
examples/check_dw_pdt.py      : b'OK!'
examples/wid_specgram.py      : b'p, _type, fmod             ^ SyntaxError: invalid syntax '
examples/cluster_DA.py        : b'OK!'
examples/check_dw_pdt1.py     : b'OK!'
examples/igetfile.py          : b"ule>     dat.plot() NameError: name 'dat' is not defined "
examples/mode_identify_example_2012.py: b"orbar=colorbar NameError: name 'colorbar' is not defined "
examples/plot_signals_trivial.py: b'OK!'
examples/small_65.py          : b'OK!'
examples/ne_profile_preprocess.py: b'                           ^ SyntaxError: invalid syntax '
examples/small_300.py         : b'OK!'
examples/fft_magnitude_extraction_LHD.py: b"   ^ SyntaxError: Missing parentheses in call to 'print' "
examples/merge_basic_diagnostics.py: b"rror: [Errno 2] No such file or directory: 'DA_file.npz' "
examples/clean_up_pyfusion_text_mp.py: b'OK!'
examples/plot_svd_old.py      : b'                           ^ SyntaxError: invalid syntax '
examples/merge_basic_HJ_diagnostics.py: b"mport gethjdata ImportError: No module named 'gethjdata' "
examples/svd_simple_rotator.py: b'hronof)))                  ^ SyntaxError: invalid syntax '
examples/test_check_buttons1.py: b'OK!'
examples/testall.py           : b"wer() != 'n': NameError: name 'raw_input' is not defined "
examples/save_to_local.py     : b") TypeError: type Signal doesn't define __round__ method "
examples/density_cluster.py   : b"AttributeError: 'range' object has no attribute 'extend' "
examples/fonts.py             : b'OK!'
examples/h1_mode_identify.py  : b'amples/cluster_info.py     ^ SyntaxError: invalid syntax '
examples/prepfs_range_sep.py  : b'error: the following arguments are required: output_path '
examples/read_DF.py           : b"mport tables as tb ImportError: No module named 'tables' "
examples/test_check_buttons.py: b'OK!'
examples/__init__.py          : b'OK!'
examples/new_mode_identify_script_0.py: b"# need to adjust NameError: name 'arange' is not defined "
examples/channel_angle_games.py: b'OK!'
examples/many_specs.py        : b"   ^ SyntaxError: Missing parentheses in call to 'print' "
examples/gen_fs_general.py    : b'amed \'mdsdcl\'",)   (shot,sys.exc_info()[1].__repr__()))) '
examples/get_kappa.py         : b"port__(name, gbls) ImportError: No module named 'mdsdcl' "
examples/flip_phase.py        : b"b_0']))[0];len(wnan) NameError: name 'dd' is not defined "
examples/plot_text_pyfusion.py: b'                           ^ SyntaxError: invalid syntax '
examples/check_polarity_flips.py: b'OK!'
examples/play_with_fft_signals.py: b'OK!'
examples/prepfs_range_mp.py   : b" AttributeError: 'module' object has no attribute 'izip' "
64 errors out of 90  (3.5 was 65 )
